[Tue Jan 11 11:45:35.573739 2022] [cgi:error] [pid 6322] [client 127.0.0.1:50318] AH01215: Traceback (most recent call last):: /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 11:45:35.574067 2022] [cgi:error] [pid 6322] [client 127.0.0.1:50318] AH01215:   File "/home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py", line 24, in <module>: /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 11:45:35.574136 2022] [cgi:error] [pid 6322] [client 127.0.0.1:50318] AH01215:     print(json.dumps(response)): /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 11:45:35.574187 2022] [cgi:error] [pid 6322] [client 127.0.0.1:50318] AH01215: NameError: name 'json' is not defined: /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 11:45:42.523504 2022] [cgi:error] [pid 6323] [client 127.0.0.1:50320] AH01215: Traceback (most recent call last):: /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 11:45:42.523632 2022] [cgi:error] [pid 6323] [client 127.0.0.1:50320] AH01215:   File "/home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py", line 24, in <module>: /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 11:45:42.523660 2022] [cgi:error] [pid 6323] [client 127.0.0.1:50320] AH01215:     print(json.dumps(response)): /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 11:45:42.523688 2022] [cgi:error] [pid 6323] [client 127.0.0.1:50320] AH01215: NameError: name 'json' is not defined: /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 11:45:59.897064 2022] [cgi:error] [pid 6324] [client 127.0.0.1:50322] AH01215: Traceback (most recent call last):: /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 11:45:59.897191 2022] [cgi:error] [pid 6324] [client 127.0.0.1:50322] AH01215:   File "/home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py", line 24, in <module>: /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 11:45:59.897218 2022] [cgi:error] [pid 6324] [client 127.0.0.1:50322] AH01215:     print(json.dumps(response)): /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 11:45:59.897247 2022] [cgi:error] [pid 6324] [client 127.0.0.1:50322] AH01215: NameError: name 'json' is not defined: /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 11:54:50.427849 2022] [cgi:error] [pid 6460] [client 127.0.0.1:50362] AH01215: Traceback (most recent call last):: /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 11:54:50.427999 2022] [cgi:error] [pid 6460] [client 127.0.0.1:50362] AH01215:   File "/home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py", line 24, in <module>: /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 11:54:50.428034 2022] [cgi:error] [pid 6460] [client 127.0.0.1:50362] AH01215:     print(json.dumps(response)): /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 11:54:50.428070 2022] [cgi:error] [pid 6460] [client 127.0.0.1:50362] AH01215: NameError: name 'json' is not defined: /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 12:28:17.709424 2022] [cgi:error] [pid 6320] [client 127.0.0.1:53086] malformed header from script 'xfc.py': Bad header: hola esto es prueba
[Tue Jan 11 12:29:14.652811 2022] [cgi:error] [pid 6321] [client 127.0.0.1:53090] malformed header from script 'xfc.py': Bad header: hola esto es prueba
[Tue Jan 11 14:33:00.445016 2022] [cgi:error] [pid 6323] [client 127.0.0.1:53204] AH01215: Traceback (most recent call last):: /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 14:33:00.445182 2022] [cgi:error] [pid 6323] [client 127.0.0.1:53204] AH01215:   File "/home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py", line 17, in <module>: /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 14:33:00.445212 2022] [cgi:error] [pid 6323] [client 127.0.0.1:53204] AH01215:     getattr(controller, RESOURCE)(): /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 14:33:00.445308 2022] [cgi:error] [pid 6323] [client 127.0.0.1:53204] AH01215:   File "/home/daniel/Proyectos/MostPlayed/rootsystem/application/modules/artist.py", line 35, in ver: /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 14:33:00.445331 2022] [cgi:error] [pid 6323] [client 127.0.0.1:53204] AH01215:     show(vars(self.model)): /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 14:33:00.445360 2022] [cgi:error] [pid 6323] [client 127.0.0.1:53204] AH01215: NameError: name 'show' is not defined: /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 14:33:00.455838 2022] [cgi:error] [pid 6323] [client 127.0.0.1:53204] End of script output before headers: xfc.py
[Tue Jan 11 14:59:08.742130 2022] [cgi:error] [pid 6460] [client 127.0.0.1:53338] AH01215: Traceback (most recent call last):: /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 14:59:08.742277 2022] [cgi:error] [pid 6460] [client 127.0.0.1:53338] AH01215:   File "/home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py", line 17, in <module>: /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 14:59:08.742307 2022] [cgi:error] [pid 6460] [client 127.0.0.1:53338] AH01215:     getattr(controller, RESOURCE)(): /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 14:59:08.742364 2022] [cgi:error] [pid 6460] [client 127.0.0.1:53338] AH01215:   File "/home/daniel/Proyectos/MostPlayed/rootsystem/application/modules/song.py", line 78, in ver: /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 14:59:08.742401 2022] [cgi:error] [pid 6460] [client 127.0.0.1:53338] AH01215:     show_json(APIRESTFul(self.model).json_data): /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 14:59:08.742457 2022] [cgi:error] [pid 6460] [client 127.0.0.1:53338] AH01215:   File "/home/daniel/Proyectos/MostPlayed/rootsystem/application/core/api.py", line 7, in __init__: /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 14:59:08.742490 2022] [cgi:error] [pid 6460] [client 127.0.0.1:53338] AH01215:     self.json_data = self.dict2json(vars(obj)): /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 14:59:08.742549 2022] [cgi:error] [pid 6460] [client 127.0.0.1:53338] AH01215:   File "/home/daniel/Proyectos/MostPlayed/rootsystem/application/core/api.py", line 21, in dict2json: /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 14:59:08.742593 2022] [cgi:error] [pid 6460] [client 127.0.0.1:53338] AH01215:     obj = json.dumps(self.obj2dict(obj),indent=4, sort_keys=True): /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 14:59:08.742653 2022] [cgi:error] [pid 6460] [client 127.0.0.1:53338] AH01215:   File "/home/daniel/Proyectos/MostPlayed/rootsystem/application/core/api.py", line 14, in obj2dict: /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 14:59:08.742682 2022] [cgi:error] [pid 6460] [client 127.0.0.1:53338] AH01215:     obj[propiedad] = vars(valor): /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 14:59:08.742721 2022] [cgi:error] [pid 6460] [client 127.0.0.1:53338] AH01215: TypeError: vars() argument must have __dict__ attribute: /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 14:59:08.757573 2022] [cgi:error] [pid 6460] [client 127.0.0.1:53338] End of script output before headers: xfc.py
[Tue Jan 11 15:01:44.927306 2022] [cgi:error] [pid 6321] [client 127.0.0.1:53344] AH01215: Traceback (most recent call last):: /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 15:01:44.927542 2022] [cgi:error] [pid 6321] [client 127.0.0.1:53344] AH01215:   File "/home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py", line 17, in <module>: /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 15:01:44.927604 2022] [cgi:error] [pid 6321] [client 127.0.0.1:53344] AH01215:     getattr(controller, RESOURCE)(): /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 15:01:44.927663 2022] [cgi:error] [pid 6321] [client 127.0.0.1:53344] AH01215:   File "/home/daniel/Proyectos/MostPlayed/rootsystem/application/modules/song.py", line 78, in ver: /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 15:01:44.927697 2022] [cgi:error] [pid 6321] [client 127.0.0.1:53344] AH01215:     show_json(APIRESTFul(self.model).json_data): /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 15:01:44.927771 2022] [cgi:error] [pid 6321] [client 127.0.0.1:53344] AH01215:   File "/home/daniel/Proyectos/MostPlayed/rootsystem/application/core/api.py", line 7, in __init__: /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 15:01:44.927813 2022] [cgi:error] [pid 6321] [client 127.0.0.1:53344] AH01215:     self.json_data = self.dict2json(vars(obj)): /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 15:01:44.927885 2022] [cgi:error] [pid 6321] [client 127.0.0.1:53344] AH01215:   File "/home/daniel/Proyectos/MostPlayed/rootsystem/application/core/api.py", line 21, in dict2json: /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 15:01:44.927937 2022] [cgi:error] [pid 6321] [client 127.0.0.1:53344] AH01215:     obj = json.dumps(self.obj2dict(obj),indent=4, sort_keys=True): /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 15:01:44.928010 2022] [cgi:error] [pid 6321] [client 127.0.0.1:53344] AH01215:   File "/home/daniel/Proyectos/MostPlayed/rootsystem/application/core/api.py", line 14, in obj2dict: /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 15:01:44.928043 2022] [cgi:error] [pid 6321] [client 127.0.0.1:53344] AH01215:     obj[propiedad] = vars(valor): /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 15:01:44.928090 2022] [cgi:error] [pid 6321] [client 127.0.0.1:53344] AH01215: TypeError: vars() argument must have __dict__ attribute: /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 15:01:44.942689 2022] [cgi:error] [pid 6321] [client 127.0.0.1:53344] End of script output before headers: xfc.py
[Tue Jan 11 15:02:43.894859 2022] [cgi:error] [pid 6323] [client 127.0.0.1:53350] AH01215: Traceback (most recent call last):: /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 15:02:43.895117 2022] [cgi:error] [pid 6323] [client 127.0.0.1:53350] AH01215:   File "/home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py", line 17, in <module>: /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 15:02:43.895197 2022] [cgi:error] [pid 6323] [client 127.0.0.1:53350] AH01215:     getattr(controller, RESOURCE)(): /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 15:02:43.895258 2022] [cgi:error] [pid 6323] [client 127.0.0.1:53350] AH01215:   File "/home/daniel/Proyectos/MostPlayed/rootsystem/application/modules/song.py", line 78, in ver: /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 15:02:43.895294 2022] [cgi:error] [pid 6323] [client 127.0.0.1:53350] AH01215:     show_json(APIRESTFul(self.model).json_data): /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 15:02:43.895354 2022] [cgi:error] [pid 6323] [client 127.0.0.1:53350] AH01215:   File "/home/daniel/Proyectos/MostPlayed/rootsystem/application/core/api.py", line 7, in __init__: /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 15:02:43.895390 2022] [cgi:error] [pid 6323] [client 127.0.0.1:53350] AH01215:     self.json_data = self.dict2json(vars(obj)): /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 15:02:43.895450 2022] [cgi:error] [pid 6323] [client 127.0.0.1:53350] AH01215:   File "/home/daniel/Proyectos/MostPlayed/rootsystem/application/core/api.py", line 21, in dict2json: /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 15:02:43.895526 2022] [cgi:error] [pid 6323] [client 127.0.0.1:53350] AH01215:     obj = json.dumps(self.obj2dict(obj),indent=4, sort_keys=True): /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 15:02:43.895587 2022] [cgi:error] [pid 6323] [client 127.0.0.1:53350] AH01215:   File "/home/daniel/Proyectos/MostPlayed/rootsystem/application/core/api.py", line 14, in obj2dict: /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 15:02:43.895615 2022] [cgi:error] [pid 6323] [client 127.0.0.1:53350] AH01215:     obj[propiedad] = vars(valor): /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 15:02:43.895654 2022] [cgi:error] [pid 6323] [client 127.0.0.1:53350] AH01215: TypeError: vars() argument must have __dict__ attribute: /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 15:02:43.909697 2022] [cgi:error] [pid 6323] [client 127.0.0.1:53350] End of script output before headers: xfc.py
[Tue Jan 11 15:09:11.529646 2022] [cgi:error] [pid 6460] [client 127.0.0.1:53382] AH01215: Traceback (most recent call last):: /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 15:09:11.529888 2022] [cgi:error] [pid 6460] [client 127.0.0.1:53382] AH01215:   File "/home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py", line 17, in <module>: /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 15:09:11.529962 2022] [cgi:error] [pid 6460] [client 127.0.0.1:53382] AH01215:     getattr(controller, RESOURCE)(): /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 15:09:11.530105 2022] [cgi:error] [pid 6460] [client 127.0.0.1:53382] AH01215:   File "/home/daniel/Proyectos/MostPlayed/rootsystem/application/modules/song.py", line 79, in ver: /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 15:09:11.530182 2022] [cgi:error] [pid 6460] [client 127.0.0.1:53382] AH01215:     show(json.dumps(vars(self.model))): /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 15:09:11.530286 2022] [cgi:error] [pid 6460] [client 127.0.0.1:53382] AH01215:   File "/usr/lib/python3.7/json/__init__.py", line 231, in dumps: /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 15:09:11.530363 2022] [cgi:error] [pid 6460] [client 127.0.0.1:53382] AH01215:     return _default_encoder.encode(obj): /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 15:09:11.530465 2022] [cgi:error] [pid 6460] [client 127.0.0.1:53382] AH01215:   File "/usr/lib/python3.7/json/encoder.py", line 199, in encode: /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 15:09:11.530550 2022] [cgi:error] [pid 6460] [client 127.0.0.1:53382] AH01215:     chunks = self.iterencode(o, _one_shot=True): /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 15:09:11.530661 2022] [cgi:error] [pid 6460] [client 127.0.0.1:53382] AH01215:   File "/usr/lib/python3.7/json/encoder.py", line 257, in iterencode: /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 15:09:11.530725 2022] [cgi:error] [pid 6460] [client 127.0.0.1:53382] AH01215:     return _iterencode(o, 0): /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 15:09:11.530829 2022] [cgi:error] [pid 6460] [client 127.0.0.1:53382] AH01215:   File "/usr/lib/python3.7/json/encoder.py", line 179, in default: /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 15:09:11.530928 2022] [cgi:error] [pid 6460] [client 127.0.0.1:53382] AH01215:     raise TypeError(f'Object of type {o.__class__.__name__} ': /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 15:09:11.531020 2022] [cgi:error] [pid 6460] [client 127.0.0.1:53382] AH01215: TypeError: Object of type date is not JSON serializable: /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 15:09:11.544298 2022] [cgi:error] [pid 6460] [client 127.0.0.1:53382] End of script output before headers: xfc.py
[Tue Jan 11 15:10:10.455645 2022] [cgi:error] [pid 6324] [client 127.0.0.1:53392] AH01215: Traceback (most recent call last):: /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 15:10:10.455790 2022] [cgi:error] [pid 6324] [client 127.0.0.1:53392] AH01215:   File "/home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py", line 12, in <module>: /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 15:10:10.455829 2022] [cgi:error] [pid 6324] [client 127.0.0.1:53392] AH01215:     modulo = __import__(PACKAGE, fromlist=[CONTROLLER]): /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 15:10:10.455882 2022] [cgi:error] [pid 6324] [client 127.0.0.1:53392] AH01215:   File "/home/daniel/Proyectos/MostPlayed/rootsystem/application/modules/song.py", line 80: /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 15:10:10.455900 2022] [cgi:error] [pid 6324] [client 127.0.0.1:53392] AH01215:     else:: /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 15:10:10.455915 2022] [cgi:error] [pid 6324] [client 127.0.0.1:53392] AH01215:        ^: /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 15:10:10.455938 2022] [cgi:error] [pid 6324] [client 127.0.0.1:53392] AH01215: SyntaxError: invalid syntax: /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 15:10:10.463249 2022] [cgi:error] [pid 6324] [client 127.0.0.1:53392] End of script output before headers: xfc.py
[Tue Jan 11 15:11:43.213973 2022] [cgi:error] [pid 6320] [client 127.0.0.1:53398] AH01215: Traceback (most recent call last):: /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 15:11:43.214137 2022] [cgi:error] [pid 6320] [client 127.0.0.1:53398] AH01215:   File "/home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py", line 17, in <module>: /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 15:11:43.214172 2022] [cgi:error] [pid 6320] [client 127.0.0.1:53398] AH01215:     getattr(controller, RESOURCE)(): /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 15:11:43.214244 2022] [cgi:error] [pid 6320] [client 127.0.0.1:53398] AH01215:   File "/home/daniel/Proyectos/MostPlayed/rootsystem/application/modules/song.py", line 79, in ver: /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 15:11:43.214288 2022] [cgi:error] [pid 6320] [client 127.0.0.1:53398] AH01215:     show_json(APIRESTFul(self.model).json_data): /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 15:11:43.214362 2022] [cgi:error] [pid 6320] [client 127.0.0.1:53398] AH01215:   File "/home/daniel/Proyectos/MostPlayed/rootsystem/application/core/api.py", line 7, in __init__: /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 15:11:43.214404 2022] [cgi:error] [pid 6320] [client 127.0.0.1:53398] AH01215:     self.json_data = self.dict2json(vars(obj)): /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 15:11:43.214477 2022] [cgi:error] [pid 6320] [client 127.0.0.1:53398] AH01215:   File "/home/daniel/Proyectos/MostPlayed/rootsystem/application/core/api.py", line 21, in dict2json: /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 15:11:43.214529 2022] [cgi:error] [pid 6320] [client 127.0.0.1:53398] AH01215:     obj = json.dumps(self.obj2dict(obj),indent=4, sort_keys=True): /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 15:11:43.214602 2022] [cgi:error] [pid 6320] [client 127.0.0.1:53398] AH01215:   File "/home/daniel/Proyectos/MostPlayed/rootsystem/application/core/api.py", line 14, in obj2dict: /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 15:11:43.214635 2022] [cgi:error] [pid 6320] [client 127.0.0.1:53398] AH01215:     obj[propiedad] = vars(valor): /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 15:11:43.214710 2022] [cgi:error] [pid 6320] [client 127.0.0.1:53398] AH01215: TypeError: vars() argument must have __dict__ attribute: /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 15:11:43.228781 2022] [cgi:error] [pid 6320] [client 127.0.0.1:53398] End of script output before headers: xfc.py
[Tue Jan 11 15:12:34.797458 2022] [cgi:error] [pid 6322] [client 127.0.0.1:53404] AH01215: Traceback (most recent call last):: /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 15:12:34.797609 2022] [cgi:error] [pid 6322] [client 127.0.0.1:53404] AH01215:   File "/home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py", line 17, in <module>: /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 15:12:34.797638 2022] [cgi:error] [pid 6322] [client 127.0.0.1:53404] AH01215:     getattr(controller, RESOURCE)(): /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 15:12:34.797696 2022] [cgi:error] [pid 6322] [client 127.0.0.1:53404] AH01215:   File "/home/daniel/Proyectos/MostPlayed/rootsystem/application/modules/song.py", line 79, in ver: /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 15:12:34.797734 2022] [cgi:error] [pid 6322] [client 127.0.0.1:53404] AH01215:     show_json(APIRESTFul(self.model).json_data): /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 15:12:34.797793 2022] [cgi:error] [pid 6322] [client 127.0.0.1:53404] AH01215:   File "/home/daniel/Proyectos/MostPlayed/rootsystem/application/core/api.py", line 7, in __init__: /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 15:12:34.797828 2022] [cgi:error] [pid 6322] [client 127.0.0.1:53404] AH01215:     self.json_data = self.dict2json(vars(obj)): /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 15:12:34.797889 2022] [cgi:error] [pid 6322] [client 127.0.0.1:53404] AH01215:   File "/home/daniel/Proyectos/MostPlayed/rootsystem/application/core/api.py", line 21, in dict2json: /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 15:12:34.797933 2022] [cgi:error] [pid 6322] [client 127.0.0.1:53404] AH01215:     obj = json.dumps(self.obj2dict(obj),indent=4, sort_keys=True): /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 15:12:34.797993 2022] [cgi:error] [pid 6322] [client 127.0.0.1:53404] AH01215:   File "/home/daniel/Proyectos/MostPlayed/rootsystem/application/core/api.py", line 14, in obj2dict: /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 15:12:34.798022 2022] [cgi:error] [pid 6322] [client 127.0.0.1:53404] AH01215:     obj[propiedad] = vars(valor): /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 15:12:34.798065 2022] [cgi:error] [pid 6322] [client 127.0.0.1:53404] AH01215: TypeError: vars() argument must have __dict__ attribute: /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 15:12:34.811731 2022] [cgi:error] [pid 6322] [client 127.0.0.1:53404] End of script output before headers: xfc.py
[Tue Jan 11 15:13:32.643058 2022] [cgi:error] [pid 6460] [client 127.0.0.1:53410] AH01215: Traceback (most recent call last):: /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 15:13:32.643202 2022] [cgi:error] [pid 6460] [client 127.0.0.1:53410] AH01215:   File "/home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py", line 17, in <module>: /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 15:13:32.643232 2022] [cgi:error] [pid 6460] [client 127.0.0.1:53410] AH01215:     getattr(controller, RESOURCE)(): /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 15:13:32.643291 2022] [cgi:error] [pid 6460] [client 127.0.0.1:53410] AH01215:   File "/home/daniel/Proyectos/MostPlayed/rootsystem/application/modules/song.py", line 79, in ver: /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 15:13:32.643354 2022] [cgi:error] [pid 6460] [client 127.0.0.1:53410] AH01215:     show_json(APIRESTFul(self.model).json_data): /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 15:13:32.643412 2022] [cgi:error] [pid 6460] [client 127.0.0.1:53410] AH01215:   File "/home/daniel/Proyectos/MostPlayed/rootsystem/application/core/api.py", line 7, in __init__: /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 15:13:32.643446 2022] [cgi:error] [pid 6460] [client 127.0.0.1:53410] AH01215:     self.json_data = self.dict2json(vars(obj)): /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 15:13:32.643507 2022] [cgi:error] [pid 6460] [client 127.0.0.1:53410] AH01215:   File "/home/daniel/Proyectos/MostPlayed/rootsystem/application/core/api.py", line 21, in dict2json: /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 15:13:32.643551 2022] [cgi:error] [pid 6460] [client 127.0.0.1:53410] AH01215:     obj = json.dumps(self.obj2dict(obj),indent=4, sort_keys=True): /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 15:13:32.643595 2022] [cgi:error] [pid 6460] [client 127.0.0.1:53410] AH01215:   File "/usr/lib/python3.7/json/__init__.py", line 238, in dumps: /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 15:13:32.643618 2022] [cgi:error] [pid 6460] [client 127.0.0.1:53410] AH01215:     **kw).encode(obj): /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 15:13:32.643661 2022] [cgi:error] [pid 6460] [client 127.0.0.1:53410] AH01215:   File "/usr/lib/python3.7/json/encoder.py", line 201, in encode: /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 15:13:32.643683 2022] [cgi:error] [pid 6460] [client 127.0.0.1:53410] AH01215:     chunks = list(chunks): /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 15:13:32.643728 2022] [cgi:error] [pid 6460] [client 127.0.0.1:53410] AH01215:   File "/usr/lib/python3.7/json/encoder.py", line 431, in _iterencode: /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 15:13:32.643769 2022] [cgi:error] [pid 6460] [client 127.0.0.1:53410] AH01215:     yield from _iterencode_dict(o, _current_indent_level): /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 15:13:32.643817 2022] [cgi:error] [pid 6460] [client 127.0.0.1:53410] AH01215:   File "/usr/lib/python3.7/json/encoder.py", line 405, in _iterencode_dict: /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 15:13:32.643839 2022] [cgi:error] [pid 6460] [client 127.0.0.1:53410] AH01215:     yield from chunks: /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 15:13:32.643887 2022] [cgi:error] [pid 6460] [client 127.0.0.1:53410] AH01215:   File "/usr/lib/python3.7/json/encoder.py", line 405, in _iterencode_dict: /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 15:13:32.643910 2022] [cgi:error] [pid 6460] [client 127.0.0.1:53410] AH01215:     yield from chunks: /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 15:13:32.643955 2022] [cgi:error] [pid 6460] [client 127.0.0.1:53410] AH01215:   File "/usr/lib/python3.7/json/encoder.py", line 438, in _iterencode: /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 15:13:32.643977 2022] [cgi:error] [pid 6460] [client 127.0.0.1:53410] AH01215:     o = _default(o): /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 15:13:32.644021 2022] [cgi:error] [pid 6460] [client 127.0.0.1:53410] AH01215:   File "/usr/lib/python3.7/json/encoder.py", line 179, in default: /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 15:13:32.644063 2022] [cgi:error] [pid 6460] [client 127.0.0.1:53410] AH01215:     raise TypeError(f'Object of type {o.__class__.__name__} ': /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 15:13:32.644103 2022] [cgi:error] [pid 6460] [client 127.0.0.1:53410] AH01215: TypeError: Object of type bytes is not JSON serializable: /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 15:13:32.656387 2022] [cgi:error] [pid 6460] [client 127.0.0.1:53410] End of script output before headers: xfc.py
[Tue Jan 11 15:13:52.507667 2022] [cgi:error] [pid 6324] [client 127.0.0.1:53412] AH01215: Traceback (most recent call last):: /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 15:13:52.507808 2022] [cgi:error] [pid 6324] [client 127.0.0.1:53412] AH01215:   File "/home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py", line 17, in <module>: /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 15:13:52.507836 2022] [cgi:error] [pid 6324] [client 127.0.0.1:53412] AH01215:     getattr(controller, RESOURCE)(): /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 15:13:52.507894 2022] [cgi:error] [pid 6324] [client 127.0.0.1:53412] AH01215:   File "/home/daniel/Proyectos/MostPlayed/rootsystem/application/modules/song.py", line 79, in ver: /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 15:13:52.507930 2022] [cgi:error] [pid 6324] [client 127.0.0.1:53412] AH01215:     show_json(APIRESTFul(self.model).json_data): /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 15:13:52.507990 2022] [cgi:error] [pid 6324] [client 127.0.0.1:53412] AH01215:   File "/home/daniel/Proyectos/MostPlayed/rootsystem/application/core/api.py", line 7, in __init__: /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 15:13:52.508025 2022] [cgi:error] [pid 6324] [client 127.0.0.1:53412] AH01215:     self.json_data = self.dict2json(vars(obj)): /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 15:13:52.508084 2022] [cgi:error] [pid 6324] [client 127.0.0.1:53412] AH01215:   File "/home/daniel/Proyectos/MostPlayed/rootsystem/application/core/api.py", line 21, in dict2json: /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 15:13:52.508127 2022] [cgi:error] [pid 6324] [client 127.0.0.1:53412] AH01215:     obj = json.dumps(self.obj2dict(obj),indent=4, sort_keys=True): /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 15:13:52.508172 2022] [cgi:error] [pid 6324] [client 127.0.0.1:53412] AH01215:   File "/usr/lib/python3.7/json/__init__.py", line 238, in dumps: /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 15:13:52.508195 2022] [cgi:error] [pid 6324] [client 127.0.0.1:53412] AH01215:     **kw).encode(obj): /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 15:13:52.508253 2022] [cgi:error] [pid 6324] [client 127.0.0.1:53412] AH01215:   File "/usr/lib/python3.7/json/encoder.py", line 201, in encode: /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 15:13:52.508282 2022] [cgi:error] [pid 6324] [client 127.0.0.1:53412] AH01215:     chunks = list(chunks): /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 15:13:52.508327 2022] [cgi:error] [pid 6324] [client 127.0.0.1:53412] AH01215:   File "/usr/lib/python3.7/json/encoder.py", line 431, in _iterencode: /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 15:13:52.508368 2022] [cgi:error] [pid 6324] [client 127.0.0.1:53412] AH01215:     yield from _iterencode_dict(o, _current_indent_level): /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 15:13:52.508416 2022] [cgi:error] [pid 6324] [client 127.0.0.1:53412] AH01215:   File "/usr/lib/python3.7/json/encoder.py", line 405, in _iterencode_dict: /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 15:13:52.508439 2022] [cgi:error] [pid 6324] [client 127.0.0.1:53412] AH01215:     yield from chunks: /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 15:13:52.508486 2022] [cgi:error] [pid 6324] [client 127.0.0.1:53412] AH01215:   File "/usr/lib/python3.7/json/encoder.py", line 405, in _iterencode_dict: /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 15:13:52.508545 2022] [cgi:error] [pid 6324] [client 127.0.0.1:53412] AH01215:     yield from chunks: /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 15:13:52.508592 2022] [cgi:error] [pid 6324] [client 127.0.0.1:53412] AH01215:   File "/usr/lib/python3.7/json/encoder.py", line 438, in _iterencode: /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 15:13:52.508614 2022] [cgi:error] [pid 6324] [client 127.0.0.1:53412] AH01215:     o = _default(o): /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 15:13:52.508657 2022] [cgi:error] [pid 6324] [client 127.0.0.1:53412] AH01215:   File "/usr/lib/python3.7/json/encoder.py", line 179, in default: /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 15:13:52.508698 2022] [cgi:error] [pid 6324] [client 127.0.0.1:53412] AH01215:     raise TypeError(f'Object of type {o.__class__.__name__} ': /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 15:13:52.508738 2022] [cgi:error] [pid 6324] [client 127.0.0.1:53412] AH01215: TypeError: Object of type bytes is not JSON serializable: /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 15:13:52.519512 2022] [cgi:error] [pid 6324] [client 127.0.0.1:53412] End of script output before headers: xfc.py
[Tue Jan 11 15:14:32.962356 2022] [cgi:error] [pid 6321] [client 127.0.0.1:53420] AH01215: Traceback (most recent call last):: /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 15:14:32.962502 2022] [cgi:error] [pid 6321] [client 127.0.0.1:53420] AH01215:   File "/home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py", line 17, in <module>: /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 15:14:32.962532 2022] [cgi:error] [pid 6321] [client 127.0.0.1:53420] AH01215:     getattr(controller, RESOURCE)(): /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 15:14:32.962590 2022] [cgi:error] [pid 6321] [client 127.0.0.1:53420] AH01215:   File "/home/daniel/Proyectos/MostPlayed/rootsystem/application/modules/song.py", line 76, in ver: /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 15:14:32.962613 2022] [cgi:error] [pid 6321] [client 127.0.0.1:53420] AH01215:     self.model.select(): /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 15:14:32.962676 2022] [cgi:error] [pid 6321] [client 127.0.0.1:53420] AH01215:   File "/home/daniel/Proyectos/MostPlayed/rootsystem/application/modules/song.py", line 54, in select: /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 15:14:32.962702 2022] [cgi:error] [pid 6321] [client 127.0.0.1:53420] AH01215:     self.artist = list(obj): /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 15:14:32.962733 2022] [cgi:error] [pid 6321] [client 127.0.0.1:53420] AH01215: TypeError: 'Artist' object is not iterable: /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 15:14:32.974372 2022] [cgi:error] [pid 6321] [client 127.0.0.1:53420] End of script output before headers: xfc.py
[Tue Jan 11 15:19:54.977820 2022] [cgi:error] [pid 6322] [client 127.0.0.1:53428] AH01215: Traceback (most recent call last):: /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 15:19:54.977968 2022] [cgi:error] [pid 6322] [client 127.0.0.1:53428] AH01215:   File "/home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py", line 17, in <module>: /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 15:19:54.977998 2022] [cgi:error] [pid 6322] [client 127.0.0.1:53428] AH01215:     getattr(controller, RESOURCE)(): /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 15:19:54.978056 2022] [cgi:error] [pid 6322] [client 127.0.0.1:53428] AH01215:   File "/home/daniel/Proyectos/MostPlayed/rootsystem/application/modules/song.py", line 81, in ver: /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 15:19:54.978117 2022] [cgi:error] [pid 6322] [client 127.0.0.1:53428] AH01215:     show_json(APIRESTFul(self.model).json_data): /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 15:19:54.978178 2022] [cgi:error] [pid 6322] [client 127.0.0.1:53428] AH01215:   File "/home/daniel/Proyectos/MostPlayed/rootsystem/application/core/api.py", line 7, in __init__: /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 15:19:54.978213 2022] [cgi:error] [pid 6322] [client 127.0.0.1:53428] AH01215:     self.json_data = self.dict2json(vars(obj)): /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 15:19:54.978275 2022] [cgi:error] [pid 6322] [client 127.0.0.1:53428] AH01215:   File "/home/daniel/Proyectos/MostPlayed/rootsystem/application/core/api.py", line 21, in dict2json: /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 15:19:54.978320 2022] [cgi:error] [pid 6322] [client 127.0.0.1:53428] AH01215:     obj = json.dumps(self.obj2dict(obj),indent=4, sort_keys=True): /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 15:19:54.978379 2022] [cgi:error] [pid 6322] [client 127.0.0.1:53428] AH01215:   File "/home/daniel/Proyectos/MostPlayed/rootsystem/application/core/api.py", line 14, in obj2dict: /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 15:19:54.978407 2022] [cgi:error] [pid 6322] [client 127.0.0.1:53428] AH01215:     obj[propiedad] = vars(valor): /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 15:19:54.978446 2022] [cgi:error] [pid 6322] [client 127.0.0.1:53428] AH01215: TypeError: vars() argument must have __dict__ attribute: /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 15:19:54.991758 2022] [cgi:error] [pid 6322] [client 127.0.0.1:53428] End of script output before headers: xfc.py
[Tue Jan 11 15:20:37.716515 2022] [cgi:error] [pid 6460] [client 127.0.0.1:53430] AH01215: Traceback (most recent call last):: /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 15:20:37.716656 2022] [cgi:error] [pid 6460] [client 127.0.0.1:53430] AH01215:   File "/home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py", line 17, in <module>: /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 15:20:37.716685 2022] [cgi:error] [pid 6460] [client 127.0.0.1:53430] AH01215:     getattr(controller, RESOURCE)(): /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 15:20:37.716742 2022] [cgi:error] [pid 6460] [client 127.0.0.1:53430] AH01215:   File "/home/daniel/Proyectos/MostPlayed/rootsystem/application/modules/song.py", line 81, in ver: /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 15:20:37.716777 2022] [cgi:error] [pid 6460] [client 127.0.0.1:53430] AH01215:     show_json(APIRESTFul(self.model).json_data): /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 15:20:37.716836 2022] [cgi:error] [pid 6460] [client 127.0.0.1:53430] AH01215:   File "/home/daniel/Proyectos/MostPlayed/rootsystem/application/core/api.py", line 7, in __init__: /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 15:20:37.716870 2022] [cgi:error] [pid 6460] [client 127.0.0.1:53430] AH01215:     self.json_data = self.dict2json(vars(obj)): /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 15:20:37.716930 2022] [cgi:error] [pid 6460] [client 127.0.0.1:53430] AH01215:   File "/home/daniel/Proyectos/MostPlayed/rootsystem/application/core/api.py", line 21, in dict2json: /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 15:20:37.716975 2022] [cgi:error] [pid 6460] [client 127.0.0.1:53430] AH01215:     obj = json.dumps(self.obj2dict(obj),indent=4, sort_keys=True): /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 15:20:37.717035 2022] [cgi:error] [pid 6460] [client 127.0.0.1:53430] AH01215:   File "/home/daniel/Proyectos/MostPlayed/rootsystem/application/core/api.py", line 14, in obj2dict: /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 15:20:37.717093 2022] [cgi:error] [pid 6460] [client 127.0.0.1:53430] AH01215:     obj[propiedad] = vars(valor): /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 15:20:37.717132 2022] [cgi:error] [pid 6460] [client 127.0.0.1:53430] AH01215: TypeError: vars() argument must have __dict__ attribute: /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 15:20:37.730341 2022] [cgi:error] [pid 6460] [client 127.0.0.1:53430] End of script output before headers: xfc.py
[Tue Jan 11 15:22:21.452013 2022] [cgi:error] [pid 6321] [client 127.0.0.1:53438] AH01215: Traceback (most recent call last):: /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 15:22:21.452291 2022] [cgi:error] [pid 6321] [client 127.0.0.1:53438] AH01215:   File "/home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py", line 17, in <module>: /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 15:22:21.452323 2022] [cgi:error] [pid 6321] [client 127.0.0.1:53438] AH01215:     getattr(controller, RESOURCE)(): /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 15:22:21.452384 2022] [cgi:error] [pid 6321] [client 127.0.0.1:53438] AH01215:   File "/home/daniel/Proyectos/MostPlayed/rootsystem/application/modules/song.py", line 81, in ver: /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 15:22:21.452418 2022] [cgi:error] [pid 6321] [client 127.0.0.1:53438] AH01215:     show_json(json.dumps(vars(self.model))): /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 15:22:21.452461 2022] [cgi:error] [pid 6321] [client 127.0.0.1:53438] AH01215:   File "/usr/lib/python3.7/json/__init__.py", line 231, in dumps: /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 15:22:21.452492 2022] [cgi:error] [pid 6321] [client 127.0.0.1:53438] AH01215:     return _default_encoder.encode(obj): /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 15:22:21.452535 2022] [cgi:error] [pid 6321] [client 127.0.0.1:53438] AH01215:   File "/usr/lib/python3.7/json/encoder.py", line 199, in encode: /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 15:22:21.452570 2022] [cgi:error] [pid 6321] [client 127.0.0.1:53438] AH01215:     chunks = self.iterencode(o, _one_shot=True): /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 15:22:21.452615 2022] [cgi:error] [pid 6321] [client 127.0.0.1:53438] AH01215:   File "/usr/lib/python3.7/json/encoder.py", line 257, in iterencode: /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 15:22:21.452641 2022] [cgi:error] [pid 6321] [client 127.0.0.1:53438] AH01215:     return _iterencode(o, 0): /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 15:22:21.452685 2022] [cgi:error] [pid 6321] [client 127.0.0.1:53438] AH01215:   File "/usr/lib/python3.7/json/encoder.py", line 179, in default: /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 15:22:21.452727 2022] [cgi:error] [pid 6321] [client 127.0.0.1:53438] AH01215:     raise TypeError(f'Object of type {o.__class__.__name__} ': /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 15:22:21.452768 2022] [cgi:error] [pid 6321] [client 127.0.0.1:53438] AH01215: TypeError: Object of type bytes is not JSON serializable: /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 15:22:21.467576 2022] [cgi:error] [pid 6321] [client 127.0.0.1:53438] End of script output before headers: xfc.py
[Tue Jan 11 15:27:56.607355 2022] [cgi:error] [pid 6320] [client 127.0.0.1:53450] AH01215: Traceback (most recent call last):: /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 15:27:56.607504 2022] [cgi:error] [pid 6320] [client 127.0.0.1:53450] AH01215:   File "/home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py", line 17, in <module>: /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 15:27:56.607571 2022] [cgi:error] [pid 6320] [client 127.0.0.1:53450] AH01215:     getattr(controller, RESOURCE)(): /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 15:27:56.607632 2022] [cgi:error] [pid 6320] [client 127.0.0.1:53450] AH01215:   File "/home/daniel/Proyectos/MostPlayed/rootsystem/application/modules/song.py", line 80, in ver: /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 15:27:56.607676 2022] [cgi:error] [pid 6320] [client 127.0.0.1:53450] AH01215:     self.model.artist = APIRESTFul(self.model.artist).json_data: /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 15:27:56.607737 2022] [cgi:error] [pid 6320] [client 127.0.0.1:53450] AH01215:   File "/home/daniel/Proyectos/MostPlayed/rootsystem/application/core/api.py", line 7, in __init__: /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 15:27:56.607774 2022] [cgi:error] [pid 6320] [client 127.0.0.1:53450] AH01215:     self.json_data = self.dict2json(vars(obj)): /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 15:27:56.607835 2022] [cgi:error] [pid 6320] [client 127.0.0.1:53450] AH01215:   File "/home/daniel/Proyectos/MostPlayed/rootsystem/application/core/api.py", line 21, in dict2json: /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 15:27:56.607880 2022] [cgi:error] [pid 6320] [client 127.0.0.1:53450] AH01215:     obj = json.dumps(self.obj2dict(obj),indent=4, sort_keys=True): /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 15:27:56.607941 2022] [cgi:error] [pid 6320] [client 127.0.0.1:53450] AH01215:   File "/home/daniel/Proyectos/MostPlayed/rootsystem/application/core/api.py", line 14, in obj2dict: /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 15:27:56.607971 2022] [cgi:error] [pid 6320] [client 127.0.0.1:53450] AH01215:     obj[propiedad] = vars(valor): /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 15:27:56.608013 2022] [cgi:error] [pid 6320] [client 127.0.0.1:53450] AH01215: TypeError: vars() argument must have __dict__ attribute: /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 15:27:56.619653 2022] [cgi:error] [pid 6320] [client 127.0.0.1:53450] End of script output before headers: xfc.py
[Tue Jan 11 15:28:53.553461 2022] [cgi:error] [pid 6322] [client 127.0.0.1:53452] AH01215: Traceback (most recent call last):: /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 15:28:53.553622 2022] [cgi:error] [pid 6322] [client 127.0.0.1:53452] AH01215:   File "/home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py", line 17, in <module>: /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 15:28:53.553651 2022] [cgi:error] [pid 6322] [client 127.0.0.1:53452] AH01215:     getattr(controller, RESOURCE)(): /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 15:28:53.553708 2022] [cgi:error] [pid 6322] [client 127.0.0.1:53452] AH01215:   File "/home/daniel/Proyectos/MostPlayed/rootsystem/application/modules/song.py", line 80, in ver: /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 15:28:53.553750 2022] [cgi:error] [pid 6322] [client 127.0.0.1:53452] AH01215:     self.model.artist = json.dumps(self.model.artist): /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 15:28:53.553791 2022] [cgi:error] [pid 6322] [client 127.0.0.1:53452] AH01215:   File "/usr/lib/python3.7/json/__init__.py", line 231, in dumps: /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 15:28:53.553821 2022] [cgi:error] [pid 6322] [client 127.0.0.1:53452] AH01215:     return _default_encoder.encode(obj): /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 15:28:53.553906 2022] [cgi:error] [pid 6322] [client 127.0.0.1:53452] AH01215:   File "/usr/lib/python3.7/json/encoder.py", line 199, in encode: /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 15:28:53.553940 2022] [cgi:error] [pid 6322] [client 127.0.0.1:53452] AH01215:     chunks = self.iterencode(o, _one_shot=True): /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 15:28:53.553984 2022] [cgi:error] [pid 6322] [client 127.0.0.1:53452] AH01215:   File "/usr/lib/python3.7/json/encoder.py", line 257, in iterencode: /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 15:28:53.554009 2022] [cgi:error] [pid 6322] [client 127.0.0.1:53452] AH01215:     return _iterencode(o, 0): /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 15:28:53.554052 2022] [cgi:error] [pid 6322] [client 127.0.0.1:53452] AH01215:   File "/usr/lib/python3.7/json/encoder.py", line 179, in default: /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 15:28:53.554094 2022] [cgi:error] [pid 6322] [client 127.0.0.1:53452] AH01215:     raise TypeError(f'Object of type {o.__class__.__name__} ': /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 15:28:53.554134 2022] [cgi:error] [pid 6322] [client 127.0.0.1:53452] AH01215: TypeError: Object of type Artist is not JSON serializable: /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 15:28:53.565735 2022] [cgi:error] [pid 6322] [client 127.0.0.1:53452] End of script output before headers: xfc.py
[Tue Jan 11 15:29:29.430514 2022] [cgi:error] [pid 6460] [client 127.0.0.1:53456] AH01215: Traceback (most recent call last):: /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 15:29:29.430682 2022] [cgi:error] [pid 6460] [client 127.0.0.1:53456] AH01215:   File "/home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py", line 17, in <module>: /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 15:29:29.430714 2022] [cgi:error] [pid 6460] [client 127.0.0.1:53456] AH01215:     getattr(controller, RESOURCE)(): /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 15:29:29.430778 2022] [cgi:error] [pid 6460] [client 127.0.0.1:53456] AH01215:   File "/home/daniel/Proyectos/MostPlayed/rootsystem/application/modules/song.py", line 80, in ver: /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 15:29:29.430810 2022] [cgi:error] [pid 6460] [client 127.0.0.1:53456] AH01215:     self.model.artist = json.dumps(obj_dicc): /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 15:29:29.430850 2022] [cgi:error] [pid 6460] [client 127.0.0.1:53456] AH01215:   File "/usr/lib/python3.7/json/__init__.py", line 231, in dumps: /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 15:29:29.430881 2022] [cgi:error] [pid 6460] [client 127.0.0.1:53456] AH01215:     return _default_encoder.encode(obj): /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 15:29:29.430925 2022] [cgi:error] [pid 6460] [client 127.0.0.1:53456] AH01215:   File "/usr/lib/python3.7/json/encoder.py", line 199, in encode: /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 15:29:29.430961 2022] [cgi:error] [pid 6460] [client 127.0.0.1:53456] AH01215:     chunks = self.iterencode(o, _one_shot=True): /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 15:29:29.431006 2022] [cgi:error] [pid 6460] [client 127.0.0.1:53456] AH01215:   File "/usr/lib/python3.7/json/encoder.py", line 257, in iterencode: /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 15:29:29.431034 2022] [cgi:error] [pid 6460] [client 127.0.0.1:53456] AH01215:     return _iterencode(o, 0): /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 15:29:29.431078 2022] [cgi:error] [pid 6460] [client 127.0.0.1:53456] AH01215:   File "/usr/lib/python3.7/json/encoder.py", line 179, in default: /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 15:29:29.431179 2022] [cgi:error] [pid 6460] [client 127.0.0.1:53456] AH01215:     raise TypeError(f'Object of type {o.__class__.__name__} ': /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 15:29:29.431221 2022] [cgi:error] [pid 6460] [client 127.0.0.1:53456] AH01215: TypeError: Object of type bytes is not JSON serializable: /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 15:29:29.441775 2022] [cgi:error] [pid 6460] [client 127.0.0.1:53456] End of script output before headers: xfc.py
[Tue Jan 11 15:33:14.961440 2022] [cgi:error] [pid 6321] [client 127.0.0.1:53464] AH01215: Traceback (most recent call last):: /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 15:33:14.961585 2022] [cgi:error] [pid 6321] [client 127.0.0.1:53464] AH01215:   File "/home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py", line 17, in <module>: /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 15:33:14.961614 2022] [cgi:error] [pid 6321] [client 127.0.0.1:53464] AH01215:     getattr(controller, RESOURCE)(): /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 15:33:14.961673 2022] [cgi:error] [pid 6321] [client 127.0.0.1:53464] AH01215:   File "/home/daniel/Proyectos/MostPlayed/rootsystem/application/modules/song.py", line 81, in ver: /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 15:33:14.961709 2022] [cgi:error] [pid 6321] [client 127.0.0.1:53464] AH01215:     show_json(APIRESTFul(self.model).json_data): /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 15:33:14.961770 2022] [cgi:error] [pid 6321] [client 127.0.0.1:53464] AH01215:   File "/home/daniel/Proyectos/MostPlayed/rootsystem/application/core/api.py", line 7, in __init__: /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 15:33:14.961806 2022] [cgi:error] [pid 6321] [client 127.0.0.1:53464] AH01215:     self.json_data = self.dict2json(vars(obj)): /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 15:33:14.961866 2022] [cgi:error] [pid 6321] [client 127.0.0.1:53464] AH01215:   File "/home/daniel/Proyectos/MostPlayed/rootsystem/application/core/api.py", line 21, in dict2json: /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 15:33:14.961908 2022] [cgi:error] [pid 6321] [client 127.0.0.1:53464] AH01215:     obj = json.dumps(self.obj2dict(obj),indent=4, sort_keys=True): /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 15:33:14.961966 2022] [cgi:error] [pid 6321] [client 127.0.0.1:53464] AH01215:   File "/home/daniel/Proyectos/MostPlayed/rootsystem/application/core/api.py", line 14, in obj2dict: /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 15:33:14.961996 2022] [cgi:error] [pid 6321] [client 127.0.0.1:53464] AH01215:     obj[propiedad] = vars(valor): /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 15:33:14.962036 2022] [cgi:error] [pid 6321] [client 127.0.0.1:53464] AH01215: TypeError: vars() argument must have __dict__ attribute: /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 15:33:14.973969 2022] [cgi:error] [pid 6321] [client 127.0.0.1:53464] End of script output before headers: xfc.py
[Tue Jan 11 15:34:33.775519 2022] [cgi:error] [pid 6323] [client 127.0.0.1:53468] AH01215: Traceback (most recent call last):: /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 15:34:33.775660 2022] [cgi:error] [pid 6323] [client 127.0.0.1:53468] AH01215:   File "/home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py", line 17, in <module>: /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 15:34:33.775689 2022] [cgi:error] [pid 6323] [client 127.0.0.1:53468] AH01215:     getattr(controller, RESOURCE)(): /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 15:34:33.775782 2022] [cgi:error] [pid 6323] [client 127.0.0.1:53468] AH01215:   File "/home/daniel/Proyectos/MostPlayed/rootsystem/application/modules/song.py", line 81, in ver: /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 15:34:33.775815 2022] [cgi:error] [pid 6323] [client 127.0.0.1:53468] AH01215:     show_json(json.dumps(vars(self.model))): /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 15:34:33.775857 2022] [cgi:error] [pid 6323] [client 127.0.0.1:53468] AH01215:   File "/usr/lib/python3.7/json/__init__.py", line 231, in dumps: /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 15:34:33.775888 2022] [cgi:error] [pid 6323] [client 127.0.0.1:53468] AH01215:     return _default_encoder.encode(obj): /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 15:34:33.775930 2022] [cgi:error] [pid 6323] [client 127.0.0.1:53468] AH01215:   File "/usr/lib/python3.7/json/encoder.py", line 199, in encode: /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 15:34:33.775965 2022] [cgi:error] [pid 6323] [client 127.0.0.1:53468] AH01215:     chunks = self.iterencode(o, _one_shot=True): /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 15:34:33.776010 2022] [cgi:error] [pid 6323] [client 127.0.0.1:53468] AH01215:   File "/usr/lib/python3.7/json/encoder.py", line 257, in iterencode: /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 15:34:33.776036 2022] [cgi:error] [pid 6323] [client 127.0.0.1:53468] AH01215:     return _iterencode(o, 0): /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 15:34:33.776079 2022] [cgi:error] [pid 6323] [client 127.0.0.1:53468] AH01215:   File "/usr/lib/python3.7/json/encoder.py", line 179, in default: /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 15:34:33.776121 2022] [cgi:error] [pid 6323] [client 127.0.0.1:53468] AH01215:     raise TypeError(f'Object of type {o.__class__.__name__} ': /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 15:34:33.776160 2022] [cgi:error] [pid 6323] [client 127.0.0.1:53468] AH01215: TypeError: Object of type bytes is not JSON serializable: /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 15:34:33.787553 2022] [cgi:error] [pid 6323] [client 127.0.0.1:53468] End of script output before headers: xfc.py
[Tue Jan 11 15:39:41.737928 2022] [cgi:error] [pid 6460] [client 127.0.0.1:53480] AH01215: Traceback (most recent call last):: /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 15:39:41.738090 2022] [cgi:error] [pid 6460] [client 127.0.0.1:53480] AH01215:   File "/home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py", line 12, in <module>: /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 15:39:41.738138 2022] [cgi:error] [pid 6460] [client 127.0.0.1:53480] AH01215:     modulo = __import__(PACKAGE, fromlist=[CONTROLLER]): /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 15:39:41.738205 2022] [cgi:error] [pid 6460] [client 127.0.0.1:53480] AH01215:   File "/home/daniel/Proyectos/MostPlayed/rootsystem/application/modules/song.py", line 89: /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 15:39:41.738220 2022] [cgi:error] [pid 6460] [client 127.0.0.1:53480] AH01215:     : /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 15:39:41.738234 2022] [cgi:error] [pid 6460] [client 127.0.0.1:53480] AH01215:     ^: /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 15:39:41.738270 2022] [cgi:error] [pid 6460] [client 127.0.0.1:53480] AH01215: SyntaxError: unexpected EOF while parsing: /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 15:39:41.755274 2022] [cgi:error] [pid 6460] [client 127.0.0.1:53480] End of script output before headers: xfc.py
[Tue Jan 11 15:39:59.133615 2022] [cgi:error] [pid 6324] [client 127.0.0.1:53482] AH01215: Traceback (most recent call last):: /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 15:39:59.133790 2022] [cgi:error] [pid 6324] [client 127.0.0.1:53482] AH01215:   File "/home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py", line 12, in <module>: /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 15:39:59.133829 2022] [cgi:error] [pid 6324] [client 127.0.0.1:53482] AH01215:     modulo = __import__(PACKAGE, fromlist=[CONTROLLER]): /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 15:39:59.133882 2022] [cgi:error] [pid 6324] [client 127.0.0.1:53482] AH01215:   File "/home/daniel/Proyectos/MostPlayed/rootsystem/application/modules/song.py", line 89: /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 15:39:59.133898 2022] [cgi:error] [pid 6324] [client 127.0.0.1:53482] AH01215:     : /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 15:39:59.133910 2022] [cgi:error] [pid 6324] [client 127.0.0.1:53482] AH01215:     ^: /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 15:39:59.133940 2022] [cgi:error] [pid 6324] [client 127.0.0.1:53482] AH01215: SyntaxError: unexpected EOF while parsing: /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 15:39:59.142296 2022] [cgi:error] [pid 6324] [client 127.0.0.1:53482] End of script output before headers: xfc.py
[Tue Jan 11 16:28:15.382309 2022] [cgi:error] [pid 6460] [client 127.0.0.1:53590] AH01215: Traceback (most recent call last):: /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 16:28:15.382507 2022] [cgi:error] [pid 6460] [client 127.0.0.1:53590] AH01215:   File "/home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py", line 17, in <module>: /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 16:28:15.382569 2022] [cgi:error] [pid 6460] [client 127.0.0.1:53590] AH01215:     getattr(controller, RESOURCE)(): /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 16:28:15.382692 2022] [cgi:error] [pid 6460] [client 127.0.0.1:53590] AH01215:   File "/home/daniel/Proyectos/MostPlayed/rootsystem/application/modules/song.py", line 88, in search_lookup: /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 16:28:15.382730 2022] [cgi:error] [pid 6460] [client 127.0.0.1:53590] AH01215:     form = FieldStorage(): /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 16:28:15.382792 2022] [cgi:error] [pid 6460] [client 127.0.0.1:53590] AH01215:   File "/usr/lib/python3.7/cgi.py", line 491, in __init__: /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 16:28:15.382827 2022] [cgi:error] [pid 6460] [client 127.0.0.1:53590] AH01215:     self.read_single(): /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 16:28:15.382887 2022] [cgi:error] [pid 6460] [client 127.0.0.1:53590] AH01215:   File "/usr/lib/python3.7/cgi.py", line 682, in read_single: /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 16:28:15.382921 2022] [cgi:error] [pid 6460] [client 127.0.0.1:53590] AH01215:     self.read_binary(): /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 16:28:15.382980 2022] [cgi:error] [pid 6460] [client 127.0.0.1:53590] AH01215:   File "/usr/lib/python3.7/cgi.py", line 704, in read_binary: /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 16:28:15.383017 2022] [cgi:error] [pid 6460] [client 127.0.0.1:53590] AH01215:     self.file.write(data): /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 16:28:15.383071 2022] [cgi:error] [pid 6460] [client 127.0.0.1:53590] AH01215: TypeError: write() argument must be str, not bytes: /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 16:28:15.412719 2022] [cgi:error] [pid 6460] [client 127.0.0.1:53590] End of script output before headers: xfc.py
[Tue Jan 11 16:29:11.949601 2022] [cgi:error] [pid 6324] [client 127.0.0.1:53592] AH01215: Traceback (most recent call last):: /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 16:29:11.949789 2022] [cgi:error] [pid 6324] [client 127.0.0.1:53592] AH01215:   File "/home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py", line 17, in <module>: /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 16:29:11.949823 2022] [cgi:error] [pid 6324] [client 127.0.0.1:53592] AH01215:     getattr(controller, RESOURCE)(): /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 16:29:11.949892 2022] [cgi:error] [pid 6324] [client 127.0.0.1:53592] AH01215:   File "/home/daniel/Proyectos/MostPlayed/rootsystem/application/modules/song.py", line 88, in search_lookup: /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 16:29:11.949917 2022] [cgi:error] [pid 6324] [client 127.0.0.1:53592] AH01215:     form = FieldStorage(): /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 16:29:11.949959 2022] [cgi:error] [pid 6324] [client 127.0.0.1:53592] AH01215:   File "/usr/lib/python3.7/cgi.py", line 491, in __init__: /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 16:29:11.949981 2022] [cgi:error] [pid 6324] [client 127.0.0.1:53592] AH01215:     self.read_single(): /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 16:29:11.950023 2022] [cgi:error] [pid 6324] [client 127.0.0.1:53592] AH01215:   File "/usr/lib/python3.7/cgi.py", line 682, in read_single: /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 16:29:11.950045 2022] [cgi:error] [pid 6324] [client 127.0.0.1:53592] AH01215:     self.read_binary(): /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 16:29:11.950087 2022] [cgi:error] [pid 6324] [client 127.0.0.1:53592] AH01215:   File "/usr/lib/python3.7/cgi.py", line 704, in read_binary: /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 16:29:11.950111 2022] [cgi:error] [pid 6324] [client 127.0.0.1:53592] AH01215:     self.file.write(data): /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 16:29:11.950148 2022] [cgi:error] [pid 6324] [client 127.0.0.1:53592] AH01215: TypeError: write() argument must be str, not bytes: /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 16:29:11.975234 2022] [cgi:error] [pid 6324] [client 127.0.0.1:53592] End of script output before headers: xfc.py
[Tue Jan 11 16:30:16.262537 2022] [cgi:error] [pid 6321] [client 127.0.0.1:53594] AH01215: Traceback (most recent call last):: /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 16:30:16.262691 2022] [cgi:error] [pid 6321] [client 127.0.0.1:53594] AH01215:   File "/home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py", line 17, in <module>: /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 16:30:16.262722 2022] [cgi:error] [pid 6321] [client 127.0.0.1:53594] AH01215:     getattr(controller, RESOURCE)(): /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 16:30:16.262785 2022] [cgi:error] [pid 6321] [client 127.0.0.1:53594] AH01215:   File "/home/daniel/Proyectos/MostPlayed/rootsystem/application/modules/song.py", line 88, in search_lookup: /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 16:30:16.262808 2022] [cgi:error] [pid 6321] [client 127.0.0.1:53594] AH01215:     form = FieldStorage(): /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 16:30:16.262846 2022] [cgi:error] [pid 6321] [client 127.0.0.1:53594] AH01215:   File "/usr/lib/python3.7/cgi.py", line 491, in __init__: /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 16:30:16.262867 2022] [cgi:error] [pid 6321] [client 127.0.0.1:53594] AH01215:     self.read_single(): /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 16:30:16.262908 2022] [cgi:error] [pid 6321] [client 127.0.0.1:53594] AH01215:   File "/usr/lib/python3.7/cgi.py", line 682, in read_single: /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 16:30:16.262965 2022] [cgi:error] [pid 6321] [client 127.0.0.1:53594] AH01215:     self.read_binary(): /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 16:30:16.263005 2022] [cgi:error] [pid 6321] [client 127.0.0.1:53594] AH01215:   File "/usr/lib/python3.7/cgi.py", line 704, in read_binary: /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 16:30:16.263026 2022] [cgi:error] [pid 6321] [client 127.0.0.1:53594] AH01215:     self.file.write(data): /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 16:30:16.263061 2022] [cgi:error] [pid 6321] [client 127.0.0.1:53594] AH01215: TypeError: write() argument must be str, not bytes: /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 16:30:16.287920 2022] [cgi:error] [pid 6321] [client 127.0.0.1:53594] End of script output before headers: xfc.py
[Tue Jan 11 16:34:10.987746 2022] [cgi:error] [pid 6323] [client 127.0.0.1:53604] AH01215: Traceback (most recent call last):: /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 16:34:10.987879 2022] [cgi:error] [pid 6323] [client 127.0.0.1:53604] AH01215:   File "/home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py", line 17, in <module>: /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 16:34:10.987910 2022] [cgi:error] [pid 6323] [client 127.0.0.1:53604] AH01215:     getattr(controller, RESOURCE)(): /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 16:34:10.987973 2022] [cgi:error] [pid 6323] [client 127.0.0.1:53604] AH01215:   File "/home/daniel/Proyectos/MostPlayed/rootsystem/application/modules/song.py", line 88, in search_lookup: /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 16:34:10.987999 2022] [cgi:error] [pid 6323] [client 127.0.0.1:53604] AH01215:     form = FieldStorage(): /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 16:34:10.988038 2022] [cgi:error] [pid 6323] [client 127.0.0.1:53604] AH01215:   File "/usr/lib/python3.7/cgi.py", line 491, in __init__: /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 16:34:10.988059 2022] [cgi:error] [pid 6323] [client 127.0.0.1:53604] AH01215:     self.read_single(): /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 16:34:10.988098 2022] [cgi:error] [pid 6323] [client 127.0.0.1:53604] AH01215:   File "/usr/lib/python3.7/cgi.py", line 682, in read_single: /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 16:34:10.988122 2022] [cgi:error] [pid 6323] [client 127.0.0.1:53604] AH01215:     self.read_binary(): /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 16:34:10.988162 2022] [cgi:error] [pid 6323] [client 127.0.0.1:53604] AH01215:   File "/usr/lib/python3.7/cgi.py", line 704, in read_binary: /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 16:34:10.988190 2022] [cgi:error] [pid 6323] [client 127.0.0.1:53604] AH01215:     self.file.write(data): /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 16:34:10.988249 2022] [cgi:error] [pid 6323] [client 127.0.0.1:53604] AH01215: TypeError: write() argument must be str, not bytes: /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 16:34:11.023974 2022] [cgi:error] [pid 6323] [client 127.0.0.1:53604] End of script output before headers: xfc.py
[Tue Jan 11 16:42:46.869179 2022] [cgi:error] [pid 6460] [client 127.0.0.1:53624] AH01215: Traceback (most recent call last):: /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 16:42:46.869331 2022] [cgi:error] [pid 6460] [client 127.0.0.1:53624] AH01215:   File "/home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py", line 17, in <module>: /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 16:42:46.869425 2022] [cgi:error] [pid 6460] [client 127.0.0.1:53624] AH01215:     getattr(controller, RESOURCE)(): /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 16:42:46.869503 2022] [cgi:error] [pid 6460] [client 127.0.0.1:53624] AH01215:   File "/home/daniel/Proyectos/MostPlayed/rootsystem/application/modules/song.py", line 88, in search_lookup: /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 16:42:46.869541 2022] [cgi:error] [pid 6460] [client 127.0.0.1:53624] AH01215:     from urllib.scheme import parse_qs: /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 16:42:46.869598 2022] [cgi:error] [pid 6460] [client 127.0.0.1:53624] AH01215: ModuleNotFoundError: No module named 'urllib.scheme': /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 16:42:46.895892 2022] [cgi:error] [pid 6460] [client 127.0.0.1:53624] End of script output before headers: xfc.py
[Tue Jan 11 16:44:30.378231 2022] [cgi:error] [pid 6324] [client 127.0.0.1:53634] AH01215: Traceback (most recent call last):: /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 16:44:30.378397 2022] [cgi:error] [pid 6324] [client 127.0.0.1:53634] AH01215:   File "/home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py", line 17, in <module>: /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 16:44:30.378432 2022] [cgi:error] [pid 6324] [client 127.0.0.1:53634] AH01215:     getattr(controller, RESOURCE)(): /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 16:44:30.378509 2022] [cgi:error] [pid 6324] [client 127.0.0.1:53634] AH01215:   File "/home/daniel/Proyectos/MostPlayed/rootsystem/application/modules/song.py", line 88, in search_lookup: /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 16:44:30.378547 2022] [cgi:error] [pid 6324] [client 127.0.0.1:53634] AH01215:     from urllib2.scheme import parse_qs: /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 16:44:30.378590 2022] [cgi:error] [pid 6324] [client 127.0.0.1:53634] AH01215: ModuleNotFoundError: No module named 'urllib2': /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 16:44:30.404054 2022] [cgi:error] [pid 6324] [client 127.0.0.1:53634] End of script output before headers: xfc.py
[Tue Jan 11 16:45:25.857330 2022] [cgi:error] [pid 6321] [client 127.0.0.1:53642] AH01215: Traceback (most recent call last):: /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 16:45:25.857481 2022] [cgi:error] [pid 6321] [client 127.0.0.1:53642] AH01215:   File "/home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py", line 17, in <module>: /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 16:45:25.857512 2022] [cgi:error] [pid 6321] [client 127.0.0.1:53642] AH01215:     getattr(controller, RESOURCE)(): /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 16:45:25.857574 2022] [cgi:error] [pid 6321] [client 127.0.0.1:53642] AH01215:   File "/home/daniel/Proyectos/MostPlayed/rootsystem/application/modules/song.py", line 96, in search_lookup: /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 16:45:25.857597 2022] [cgi:error] [pid 6321] [client 127.0.0.1:53642] AH01215:     show(parse_qs(qs)) : /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 16:45:25.857641 2022] [cgi:error] [pid 6321] [client 127.0.0.1:53642] AH01215: NameError: name 'parse_qs' is not defined: /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 16:45:25.882474 2022] [cgi:error] [pid 6321] [client 127.0.0.1:53642] End of script output before headers: xfc.py
[Tue Jan 11 16:53:21.157229 2022] [cgi:error] [pid 6321] [client 127.0.0.1:53664] AH01215: Traceback (most recent call last):: /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 16:53:21.157375 2022] [cgi:error] [pid 6321] [client 127.0.0.1:53664] AH01215:   File "/home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py", line 17, in <module>: /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 16:53:21.157437 2022] [cgi:error] [pid 6321] [client 127.0.0.1:53664] AH01215:     getattr(controller, RESOURCE)(): /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 16:53:21.157500 2022] [cgi:error] [pid 6321] [client 127.0.0.1:53664] AH01215:   File "/home/daniel/Proyectos/MostPlayed/rootsystem/application/modules/song.py", line 92, in search_lookup: /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 16:53:21.157535 2022] [cgi:error] [pid 6321] [client 127.0.0.1:53664] AH01215:     resultado = SongHelper().get_name(kword): /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 16:53:21.157598 2022] [cgi:error] [pid 6321] [client 127.0.0.1:53664] AH01215:   File "/home/daniel/Proyectos/MostPlayed/rootsystem/application/modules/song.py", line 103, in get_name: /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 16:53:21.157627 2022] [cgi:error] [pid 6321] [client 127.0.0.1:53664] AH01215:     WHERE name={}""".format(name): /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 16:53:21.157657 2022] [cgi:error] [pid 6321] [client 127.0.0.1:53664] AH01215: NameError: name 'name' is not defined: /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 16:53:21.176986 2022] [cgi:error] [pid 6321] [client 127.0.0.1:53664] End of script output before headers: xfc.py
[Tue Jan 11 16:53:48.933433 2022] [cgi:error] [pid 6323] [client 127.0.0.1:53666] AH01215: Traceback (most recent call last):: /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 16:53:48.933628 2022] [cgi:error] [pid 6323] [client 127.0.0.1:53666] AH01215:   File "/home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py", line 17, in <module>: /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 16:53:48.933669 2022] [cgi:error] [pid 6323] [client 127.0.0.1:53666] AH01215:     getattr(controller, RESOURCE)(): /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 16:53:48.933784 2022] [cgi:error] [pid 6323] [client 127.0.0.1:53666] AH01215:   File "/home/daniel/Proyectos/MostPlayed/rootsystem/application/modules/song.py", line 92, in search_lookup: /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 16:53:48.933835 2022] [cgi:error] [pid 6323] [client 127.0.0.1:53666] AH01215:     resultado = SongHelper().get_name(kword): /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 16:53:48.933927 2022] [cgi:error] [pid 6323] [client 127.0.0.1:53666] AH01215:   File "/home/daniel/Proyectos/MostPlayed/rootsystem/application/modules/song.py", line 104, in get_name: /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 16:53:48.933974 2022] [cgi:error] [pid 6323] [client 127.0.0.1:53666] AH01215:     resultado = DBQuery().execute(sql): /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 16:53:48.934061 2022] [cgi:error] [pid 6323] [client 127.0.0.1:53666] AH01215:   File "/home/daniel/Proyectos/MostPlayed/rootsystem/application/core/db.py", line 17, in execute: /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 16:53:48.934100 2022] [cgi:error] [pid 6323] [client 127.0.0.1:53666] AH01215:     cursor.execute(self.sql): /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 16:53:48.934174 2022] [cgi:error] [pid 6323] [client 127.0.0.1:53666] AH01215:   File "/usr/lib/python3/dist-packages/MySQLdb/cursors.py", line 250, in execute: /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 16:53:48.934219 2022] [cgi:error] [pid 6323] [client 127.0.0.1:53666] AH01215:     self.errorhandler(self, exc, value): /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 16:53:48.934305 2022] [cgi:error] [pid 6323] [client 127.0.0.1:53666] AH01215:   File "/usr/lib/python3/dist-packages/MySQLdb/connections.py", line 50, in defaulterrorhandler: /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 16:53:48.934378 2022] [cgi:error] [pid 6323] [client 127.0.0.1:53666] AH01215:     raise errorvalue: /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 16:53:48.934450 2022] [cgi:error] [pid 6323] [client 127.0.0.1:53666] AH01215:   File "/usr/lib/python3/dist-packages/MySQLdb/cursors.py", line 247, in execute: /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 16:53:48.934487 2022] [cgi:error] [pid 6323] [client 127.0.0.1:53666] AH01215:     res = self._query(query): /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 16:53:48.934558 2022] [cgi:error] [pid 6323] [client 127.0.0.1:53666] AH01215:   File "/usr/lib/python3/dist-packages/MySQLdb/cursors.py", line 411, in _query: /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 16:53:48.934596 2022] [cgi:error] [pid 6323] [client 127.0.0.1:53666] AH01215:     rowcount = self._do_query(q): /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 16:53:48.934672 2022] [cgi:error] [pid 6323] [client 127.0.0.1:53666] AH01215:   File "/usr/lib/python3/dist-packages/MySQLdb/cursors.py", line 374, in _do_query: /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 16:53:48.934701 2022] [cgi:error] [pid 6323] [client 127.0.0.1:53666] AH01215:     db.query(q): /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 16:53:48.934774 2022] [cgi:error] [pid 6323] [client 127.0.0.1:53666] AH01215:   File "/usr/lib/python3/dist-packages/MySQLdb/connections.py", line 292, in query: /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 16:53:48.934819 2022] [cgi:error] [pid 6323] [client 127.0.0.1:53666] AH01215:     _mysql.connection.query(self, query): /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 16:53:48.934978 2022] [cgi:error] [pid 6323] [client 127.0.0.1:53666] AH01215: _mysql_exceptions.ProgrammingError: (1064, "You have an error in your SQL syntax; check the manual that corresponds to your MariaDB server version for the right syntax to use near 'Lights' at line 3"): /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 16:53:48.958194 2022] [cgi:error] [pid 6323] [client 127.0.0.1:53666] End of script output before headers: xfc.py
[Tue Jan 11 16:54:29.615375 2022] [cgi:error] [pid 6320] [client 127.0.0.1:53668] End of script output before headers: xfc.py
[Tue Jan 11 16:55:30.367805 2022] [cgi:error] [pid 6322] [client 127.0.0.1:53674] AH01215: Traceback (most recent call last):: /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 16:55:30.367953 2022] [cgi:error] [pid 6322] [client 127.0.0.1:53674] AH01215:   File "/home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py", line 17, in <module>: /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 16:55:30.367981 2022] [cgi:error] [pid 6322] [client 127.0.0.1:53674] AH01215:     getattr(controller, RESOURCE)(): /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 16:55:30.368044 2022] [cgi:error] [pid 6322] [client 127.0.0.1:53674] AH01215:   File "/home/daniel/Proyectos/MostPlayed/rootsystem/application/modules/song.py", line 90, in search_lookup: /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 16:55:30.368070 2022] [cgi:error] [pid 6322] [client 127.0.0.1:53674] AH01215:     j = json.loads(datos) : /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 16:55:30.368111 2022] [cgi:error] [pid 6322] [client 127.0.0.1:53674] AH01215:   File "/usr/lib/python3.7/json/__init__.py", line 341, in loads: /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 16:55:30.368159 2022] [cgi:error] [pid 6322] [client 127.0.0.1:53674] AH01215:     raise TypeError(f'the JSON object must be str, bytes or bytearray, ': /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 16:55:30.368262 2022] [cgi:error] [pid 6322] [client 127.0.0.1:53674] AH01215: TypeError: the JSON object must be str, bytes or bytearray, not dict: /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 16:55:30.388430 2022] [cgi:error] [pid 6322] [client 127.0.0.1:53674] End of script output before headers: xfc.py
[Tue Jan 11 16:56:56.101025 2022] [cgi:error] [pid 6460] [client 127.0.0.1:53678] End of script output before headers: xfc.py
[Tue Jan 11 16:57:29.828869 2022] [cgi:error] [pid 6324] [client 127.0.0.1:53680] End of script output before headers: xfc.py
[Tue Jan 11 17:01:34.401742 2022] [cgi:error] [pid 6321] [client 127.0.0.1:53682] End of script output before headers: xfc.py
[Tue Jan 11 17:05:14.925221 2022] [cgi:error] [pid 6323] [client 127.0.0.1:53696] AH01215: Traceback (most recent call last):: /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 17:05:14.925376 2022] [cgi:error] [pid 6323] [client 127.0.0.1:53696] AH01215:   File "/home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py", line 17, in <module>: /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 17:05:14.925408 2022] [cgi:error] [pid 6323] [client 127.0.0.1:53696] AH01215:     getattr(controller, RESOURCE)(): /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 17:05:14.925482 2022] [cgi:error] [pid 6323] [client 127.0.0.1:53696] AH01215:   File "/home/daniel/Proyectos/MostPlayed/rootsystem/application/modules/song.py", line 89, in search_lookup: /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 17:05:14.925510 2022] [cgi:error] [pid 6323] [client 127.0.0.1:53696] AH01215:     json = form['data'].value: /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 17:05:14.925552 2022] [cgi:error] [pid 6323] [client 127.0.0.1:53696] AH01215:   File "/usr/lib/python3.7/cgi.py", line 534, in __getitem__: /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 17:05:14.925575 2022] [cgi:error] [pid 6323] [client 127.0.0.1:53696] AH01215:     raise KeyError(key): /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 17:05:14.925598 2022] [cgi:error] [pid 6323] [client 127.0.0.1:53696] AH01215: KeyError: 'data': /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 17:05:14.956437 2022] [cgi:error] [pid 6323] [client 127.0.0.1:53696] End of script output before headers: xfc.py
[Tue Jan 11 17:05:52.929359 2022] [cgi:error] [pid 6320] [client 127.0.0.1:53700] AH01215: Traceback (most recent call last):: /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 17:05:52.929509 2022] [cgi:error] [pid 6320] [client 127.0.0.1:53700] AH01215:   File "/home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py", line 17, in <module>: /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 17:05:52.929541 2022] [cgi:error] [pid 6320] [client 127.0.0.1:53700] AH01215:     getattr(controller, RESOURCE)(): /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 17:05:52.929604 2022] [cgi:error] [pid 6320] [client 127.0.0.1:53700] AH01215:   File "/home/daniel/Proyectos/MostPlayed/rootsystem/application/modules/song.py", line 88, in search_lookup: /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 17:05:52.929628 2022] [cgi:error] [pid 6320] [client 127.0.0.1:53700] AH01215:     form = FieldStorage(): /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 17:05:52.929665 2022] [cgi:error] [pid 6320] [client 127.0.0.1:53700] AH01215:   File "/usr/lib/python3.7/cgi.py", line 491, in __init__: /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 17:05:52.929686 2022] [cgi:error] [pid 6320] [client 127.0.0.1:53700] AH01215:     self.read_single(): /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 17:05:52.929761 2022] [cgi:error] [pid 6320] [client 127.0.0.1:53700] AH01215:   File "/usr/lib/python3.7/cgi.py", line 682, in read_single: /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 17:05:52.929783 2022] [cgi:error] [pid 6320] [client 127.0.0.1:53700] AH01215:     self.read_binary(): /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 17:05:52.929821 2022] [cgi:error] [pid 6320] [client 127.0.0.1:53700] AH01215:   File "/usr/lib/python3.7/cgi.py", line 704, in read_binary: /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 17:05:52.929843 2022] [cgi:error] [pid 6320] [client 127.0.0.1:53700] AH01215:     self.file.write(data): /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 17:05:52.929878 2022] [cgi:error] [pid 6320] [client 127.0.0.1:53700] AH01215: TypeError: write() argument must be str, not bytes: /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 17:05:52.953082 2022] [cgi:error] [pid 6320] [client 127.0.0.1:53700] End of script output before headers: xfc.py
[Tue Jan 11 17:06:13.443081 2022] [cgi:error] [pid 6322] [client 127.0.0.1:53702] AH01215: Traceback (most recent call last):: /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 17:06:13.443222 2022] [cgi:error] [pid 6322] [client 127.0.0.1:53702] AH01215:   File "/home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py", line 17, in <module>: /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 17:06:13.443251 2022] [cgi:error] [pid 6322] [client 127.0.0.1:53702] AH01215:     getattr(controller, RESOURCE)(): /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 17:06:13.443318 2022] [cgi:error] [pid 6322] [client 127.0.0.1:53702] AH01215:   File "/home/daniel/Proyectos/MostPlayed/rootsystem/application/modules/song.py", line 88, in search_lookup: /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 17:06:13.443344 2022] [cgi:error] [pid 6322] [client 127.0.0.1:53702] AH01215:     form = FieldStorage(): /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 17:06:13.443387 2022] [cgi:error] [pid 6322] [client 127.0.0.1:53702] AH01215:   File "/usr/lib/python3.7/cgi.py", line 491, in __init__: /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 17:06:13.443411 2022] [cgi:error] [pid 6322] [client 127.0.0.1:53702] AH01215:     self.read_single(): /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 17:06:13.443451 2022] [cgi:error] [pid 6322] [client 127.0.0.1:53702] AH01215:   File "/usr/lib/python3.7/cgi.py", line 682, in read_single: /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 17:06:13.443474 2022] [cgi:error] [pid 6322] [client 127.0.0.1:53702] AH01215:     self.read_binary(): /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 17:06:13.443517 2022] [cgi:error] [pid 6322] [client 127.0.0.1:53702] AH01215:   File "/usr/lib/python3.7/cgi.py", line 704, in read_binary: /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 17:06:13.443542 2022] [cgi:error] [pid 6322] [client 127.0.0.1:53702] AH01215:     self.file.write(data): /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 17:06:13.443579 2022] [cgi:error] [pid 6322] [client 127.0.0.1:53702] AH01215: TypeError: write() argument must be str, not bytes: /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 17:06:13.466508 2022] [cgi:error] [pid 6322] [client 127.0.0.1:53702] End of script output before headers: xfc.py
[Tue Jan 11 17:06:38.557369 2022] [cgi:error] [pid 6460] [client 127.0.0.1:53708] AH01215: Traceback (most recent call last):: /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 17:06:38.557522 2022] [cgi:error] [pid 6460] [client 127.0.0.1:53708] AH01215:   File "/home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py", line 17, in <module>: /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 17:06:38.557582 2022] [cgi:error] [pid 6460] [client 127.0.0.1:53708] AH01215:     getattr(controller, RESOURCE)(): /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 17:06:38.557650 2022] [cgi:error] [pid 6460] [client 127.0.0.1:53708] AH01215:   File "/home/daniel/Proyectos/MostPlayed/rootsystem/application/modules/song.py", line 88, in search_lookup: /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 17:06:38.557673 2022] [cgi:error] [pid 6460] [client 127.0.0.1:53708] AH01215:     form = FieldStorage(): /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 17:06:38.557710 2022] [cgi:error] [pid 6460] [client 127.0.0.1:53708] AH01215:   File "/usr/lib/python3.7/cgi.py", line 491, in __init__: /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 17:06:38.557731 2022] [cgi:error] [pid 6460] [client 127.0.0.1:53708] AH01215:     self.read_single(): /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 17:06:38.557769 2022] [cgi:error] [pid 6460] [client 127.0.0.1:53708] AH01215:   File "/usr/lib/python3.7/cgi.py", line 682, in read_single: /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 17:06:38.557790 2022] [cgi:error] [pid 6460] [client 127.0.0.1:53708] AH01215:     self.read_binary(): /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 17:06:38.557828 2022] [cgi:error] [pid 6460] [client 127.0.0.1:53708] AH01215:   File "/usr/lib/python3.7/cgi.py", line 704, in read_binary: /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 17:06:38.557848 2022] [cgi:error] [pid 6460] [client 127.0.0.1:53708] AH01215:     self.file.write(data): /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 17:06:38.557882 2022] [cgi:error] [pid 6460] [client 127.0.0.1:53708] AH01215: TypeError: write() argument must be str, not bytes: /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 17:06:38.581060 2022] [cgi:error] [pid 6460] [client 127.0.0.1:53708] End of script output before headers: xfc.py
[Tue Jan 11 17:06:58.834665 2022] [cgi:error] [pid 6324] [client 127.0.0.1:53710] AH01215: Traceback (most recent call last):: /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 17:06:58.834799 2022] [cgi:error] [pid 6324] [client 127.0.0.1:53710] AH01215:   File "/home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py", line 17, in <module>: /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 17:06:58.834834 2022] [cgi:error] [pid 6324] [client 127.0.0.1:53710] AH01215:     getattr(controller, RESOURCE)(): /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 17:06:58.834912 2022] [cgi:error] [pid 6324] [client 127.0.0.1:53710] AH01215:   File "/home/daniel/Proyectos/MostPlayed/rootsystem/application/modules/song.py", line 88, in search_lookup: /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 17:06:58.834940 2022] [cgi:error] [pid 6324] [client 127.0.0.1:53710] AH01215:     form = FieldStorage(): /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 17:06:58.834985 2022] [cgi:error] [pid 6324] [client 127.0.0.1:53710] AH01215:   File "/usr/lib/python3.7/cgi.py", line 491, in __init__: /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 17:06:58.835013 2022] [cgi:error] [pid 6324] [client 127.0.0.1:53710] AH01215:     self.read_single(): /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 17:06:58.835062 2022] [cgi:error] [pid 6324] [client 127.0.0.1:53710] AH01215:   File "/usr/lib/python3.7/cgi.py", line 682, in read_single: /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 17:06:58.835089 2022] [cgi:error] [pid 6324] [client 127.0.0.1:53710] AH01215:     self.read_binary(): /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 17:06:58.835165 2022] [cgi:error] [pid 6324] [client 127.0.0.1:53710] AH01215:   File "/usr/lib/python3.7/cgi.py", line 704, in read_binary: /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 17:06:58.835195 2022] [cgi:error] [pid 6324] [client 127.0.0.1:53710] AH01215:     self.file.write(data): /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 17:06:58.835238 2022] [cgi:error] [pid 6324] [client 127.0.0.1:53710] AH01215: TypeError: write() argument must be str, not bytes: /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 17:06:58.860170 2022] [cgi:error] [pid 6324] [client 127.0.0.1:53710] End of script output before headers: xfc.py
[Tue Jan 11 17:07:27.174371 2022] [cgi:error] [pid 6321] [client 127.0.0.1:53712] AH01215: Traceback (most recent call last):: /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 17:07:27.174540 2022] [cgi:error] [pid 6321] [client 127.0.0.1:53712] AH01215:   File "/home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py", line 17, in <module>: /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 17:07:27.174571 2022] [cgi:error] [pid 6321] [client 127.0.0.1:53712] AH01215:     getattr(controller, RESOURCE)(): /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 17:07:27.174639 2022] [cgi:error] [pid 6321] [client 127.0.0.1:53712] AH01215:   File "/home/daniel/Proyectos/MostPlayed/rootsystem/application/modules/song.py", line 88, in search_lookup: /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 17:07:27.174666 2022] [cgi:error] [pid 6321] [client 127.0.0.1:53712] AH01215:     form = FieldStorage(): /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 17:07:27.174708 2022] [cgi:error] [pid 6321] [client 127.0.0.1:53712] AH01215:   File "/usr/lib/python3.7/cgi.py", line 491, in __init__: /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 17:07:27.174732 2022] [cgi:error] [pid 6321] [client 127.0.0.1:53712] AH01215:     self.read_single(): /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 17:07:27.174776 2022] [cgi:error] [pid 6321] [client 127.0.0.1:53712] AH01215:   File "/usr/lib/python3.7/cgi.py", line 682, in read_single: /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 17:07:27.174800 2022] [cgi:error] [pid 6321] [client 127.0.0.1:53712] AH01215:     self.read_binary(): /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 17:07:27.174844 2022] [cgi:error] [pid 6321] [client 127.0.0.1:53712] AH01215:   File "/usr/lib/python3.7/cgi.py", line 704, in read_binary: /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 17:07:27.174870 2022] [cgi:error] [pid 6321] [client 127.0.0.1:53712] AH01215:     self.file.write(data): /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 17:07:27.174909 2022] [cgi:error] [pid 6321] [client 127.0.0.1:53712] AH01215: TypeError: write() argument must be str, not bytes: /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 17:07:27.197731 2022] [cgi:error] [pid 6321] [client 127.0.0.1:53712] End of script output before headers: xfc.py
[Tue Jan 11 17:07:52.773845 2022] [cgi:error] [pid 6323] [client 127.0.0.1:53714] AH01215: Traceback (most recent call last):: /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 17:07:52.773980 2022] [cgi:error] [pid 6323] [client 127.0.0.1:53714] AH01215:   File "/home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py", line 17, in <module>: /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 17:07:52.774015 2022] [cgi:error] [pid 6323] [client 127.0.0.1:53714] AH01215:     getattr(controller, RESOURCE)(): /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 17:07:52.774093 2022] [cgi:error] [pid 6323] [client 127.0.0.1:53714] AH01215:   File "/home/daniel/Proyectos/MostPlayed/rootsystem/application/modules/song.py", line 88, in search_lookup: /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 17:07:52.774147 2022] [cgi:error] [pid 6323] [client 127.0.0.1:53714] AH01215:     form = FieldStorage(): /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 17:07:52.774196 2022] [cgi:error] [pid 6323] [client 127.0.0.1:53714] AH01215:   File "/usr/lib/python3.7/cgi.py", line 491, in __init__: /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 17:07:52.774221 2022] [cgi:error] [pid 6323] [client 127.0.0.1:53714] AH01215:     self.read_single(): /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 17:07:52.774273 2022] [cgi:error] [pid 6323] [client 127.0.0.1:53714] AH01215:   File "/usr/lib/python3.7/cgi.py", line 682, in read_single: /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 17:07:52.774301 2022] [cgi:error] [pid 6323] [client 127.0.0.1:53714] AH01215:     self.read_binary(): /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 17:07:52.774349 2022] [cgi:error] [pid 6323] [client 127.0.0.1:53714] AH01215:   File "/usr/lib/python3.7/cgi.py", line 704, in read_binary: /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 17:07:52.774375 2022] [cgi:error] [pid 6323] [client 127.0.0.1:53714] AH01215:     self.file.write(data): /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 17:07:52.774420 2022] [cgi:error] [pid 6323] [client 127.0.0.1:53714] AH01215: TypeError: write() argument must be str, not bytes: /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 17:07:52.797953 2022] [cgi:error] [pid 6323] [client 127.0.0.1:53714] End of script output before headers: xfc.py
[Tue Jan 11 17:10:33.912074 2022] [cgi:error] [pid 6320] [client 127.0.0.1:53718] AH01215: Traceback (most recent call last):: /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 17:10:33.912253 2022] [cgi:error] [pid 6320] [client 127.0.0.1:53718] AH01215:   File "/home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py", line 17, in <module>: /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 17:10:33.912289 2022] [cgi:error] [pid 6320] [client 127.0.0.1:53718] AH01215:     getattr(controller, RESOURCE)(): /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 17:10:33.912359 2022] [cgi:error] [pid 6320] [client 127.0.0.1:53718] AH01215:   File "/home/daniel/Proyectos/MostPlayed/rootsystem/application/modules/song.py", line 88, in search_lookup: /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 17:10:33.912384 2022] [cgi:error] [pid 6320] [client 127.0.0.1:53718] AH01215:     form = FieldStorage(): /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 17:10:33.912425 2022] [cgi:error] [pid 6320] [client 127.0.0.1:53718] AH01215:   File "/usr/lib/python3.7/cgi.py", line 491, in __init__: /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 17:10:33.912449 2022] [cgi:error] [pid 6320] [client 127.0.0.1:53718] AH01215:     self.read_single(): /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 17:10:33.912491 2022] [cgi:error] [pid 6320] [client 127.0.0.1:53718] AH01215:   File "/usr/lib/python3.7/cgi.py", line 682, in read_single: /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 17:10:33.912514 2022] [cgi:error] [pid 6320] [client 127.0.0.1:53718] AH01215:     self.read_binary(): /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 17:10:33.912557 2022] [cgi:error] [pid 6320] [client 127.0.0.1:53718] AH01215:   File "/usr/lib/python3.7/cgi.py", line 704, in read_binary: /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 17:10:33.912581 2022] [cgi:error] [pid 6320] [client 127.0.0.1:53718] AH01215:     self.file.write(data): /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 17:10:33.912667 2022] [cgi:error] [pid 6320] [client 127.0.0.1:53718] AH01215: TypeError: write() argument must be str, not bytes: /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 17:10:33.942092 2022] [cgi:error] [pid 6320] [client 127.0.0.1:53718] End of script output before headers: xfc.py
[Tue Jan 11 17:11:46.777924 2022] [cgi:error] [pid 6322] [client 127.0.0.1:53720] AH01215: Traceback (most recent call last):: /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 17:11:46.778104 2022] [cgi:error] [pid 6322] [client 127.0.0.1:53720] AH01215:   File "/home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py", line 17, in <module>: /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 17:11:46.778135 2022] [cgi:error] [pid 6322] [client 127.0.0.1:53720] AH01215:     getattr(controller, RESOURCE)(): /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 17:11:46.778199 2022] [cgi:error] [pid 6322] [client 127.0.0.1:53720] AH01215:   File "/home/daniel/Proyectos/MostPlayed/rootsystem/application/modules/song.py", line 88, in search_lookup: /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 17:11:46.778224 2022] [cgi:error] [pid 6322] [client 127.0.0.1:53720] AH01215:     form = FieldStorage(): /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 17:11:46.778266 2022] [cgi:error] [pid 6322] [client 127.0.0.1:53720] AH01215:   File "/usr/lib/python3.7/cgi.py", line 491, in __init__: /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 17:11:46.778291 2022] [cgi:error] [pid 6322] [client 127.0.0.1:53720] AH01215:     self.read_single(): /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 17:11:46.778332 2022] [cgi:error] [pid 6322] [client 127.0.0.1:53720] AH01215:   File "/usr/lib/python3.7/cgi.py", line 682, in read_single: /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 17:11:46.778355 2022] [cgi:error] [pid 6322] [client 127.0.0.1:53720] AH01215:     self.read_binary(): /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 17:11:46.778396 2022] [cgi:error] [pid 6322] [client 127.0.0.1:53720] AH01215:   File "/usr/lib/python3.7/cgi.py", line 704, in read_binary: /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 17:11:46.778422 2022] [cgi:error] [pid 6322] [client 127.0.0.1:53720] AH01215:     self.file.write(data): /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 17:11:46.778460 2022] [cgi:error] [pid 6322] [client 127.0.0.1:53720] AH01215: TypeError: write() argument must be str, not bytes: /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 17:11:46.802109 2022] [cgi:error] [pid 6322] [client 127.0.0.1:53720] End of script output before headers: xfc.py
[Tue Jan 11 17:14:58.493521 2022] [cgi:error] [pid 6460] [client 127.0.0.1:53772] AH01215: Traceback (most recent call last):: /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 17:14:58.493685 2022] [cgi:error] [pid 6460] [client 127.0.0.1:53772] AH01215:   File "/home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py", line 17, in <module>: /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 17:14:58.493718 2022] [cgi:error] [pid 6460] [client 127.0.0.1:53772] AH01215:     getattr(controller, RESOURCE)(): /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 17:14:58.493786 2022] [cgi:error] [pid 6460] [client 127.0.0.1:53772] AH01215:   File "/home/daniel/Proyectos/MostPlayed/rootsystem/application/modules/song.py", line 88, in search_lookup: /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 17:14:58.493811 2022] [cgi:error] [pid 6460] [client 127.0.0.1:53772] AH01215:     form = FieldStorage(): /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 17:14:58.493852 2022] [cgi:error] [pid 6460] [client 127.0.0.1:53772] AH01215:   File "/usr/lib/python3.7/cgi.py", line 491, in __init__: /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 17:14:58.493911 2022] [cgi:error] [pid 6460] [client 127.0.0.1:53772] AH01215:     self.read_single(): /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 17:14:58.493953 2022] [cgi:error] [pid 6460] [client 127.0.0.1:53772] AH01215:   File "/usr/lib/python3.7/cgi.py", line 682, in read_single: /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 17:14:58.493975 2022] [cgi:error] [pid 6460] [client 127.0.0.1:53772] AH01215:     self.read_binary(): /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 17:14:58.494017 2022] [cgi:error] [pid 6460] [client 127.0.0.1:53772] AH01215:   File "/usr/lib/python3.7/cgi.py", line 704, in read_binary: /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 17:14:58.494041 2022] [cgi:error] [pid 6460] [client 127.0.0.1:53772] AH01215:     self.file.write(data): /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 17:14:58.494079 2022] [cgi:error] [pid 6460] [client 127.0.0.1:53772] AH01215: TypeError: write() argument must be str, not bytes: /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 17:14:58.527679 2022] [cgi:error] [pid 6460] [client 127.0.0.1:53772] End of script output before headers: xfc.py
[Tue Jan 11 17:16:11.393251 2022] [cgi:error] [pid 6324] [client 127.0.0.1:53784] AH01215: Traceback (most recent call last):: /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 17:16:11.393404 2022] [cgi:error] [pid 6324] [client 127.0.0.1:53784] AH01215:   File "/home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py", line 17, in <module>: /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 17:16:11.393449 2022] [cgi:error] [pid 6324] [client 127.0.0.1:53784] AH01215:     getattr(controller, RESOURCE)(): /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 17:16:11.393515 2022] [cgi:error] [pid 6324] [client 127.0.0.1:53784] AH01215:   File "/home/daniel/Proyectos/MostPlayed/rootsystem/application/modules/song.py", line 88, in search_lookup: /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 17:16:11.393540 2022] [cgi:error] [pid 6324] [client 127.0.0.1:53784] AH01215:     form = FieldStorage(): /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 17:16:11.393579 2022] [cgi:error] [pid 6324] [client 127.0.0.1:53784] AH01215:   File "/usr/lib/python3.7/cgi.py", line 491, in __init__: /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 17:16:11.393602 2022] [cgi:error] [pid 6324] [client 127.0.0.1:53784] AH01215:     self.read_single(): /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 17:16:11.393643 2022] [cgi:error] [pid 6324] [client 127.0.0.1:53784] AH01215:   File "/usr/lib/python3.7/cgi.py", line 682, in read_single: /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 17:16:11.393666 2022] [cgi:error] [pid 6324] [client 127.0.0.1:53784] AH01215:     self.read_binary(): /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 17:16:11.393707 2022] [cgi:error] [pid 6324] [client 127.0.0.1:53784] AH01215:   File "/usr/lib/python3.7/cgi.py", line 704, in read_binary: /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 17:16:11.393731 2022] [cgi:error] [pid 6324] [client 127.0.0.1:53784] AH01215:     self.file.write(data): /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 17:16:11.393768 2022] [cgi:error] [pid 6324] [client 127.0.0.1:53784] AH01215: TypeError: write() argument must be str, not bytes: /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 17:16:11.417276 2022] [cgi:error] [pid 6324] [client 127.0.0.1:53784] End of script output before headers: xfc.py
[Tue Jan 11 17:25:14.929825 2022] [cgi:error] [pid 6320] [client 127.0.0.1:53802] AH01215: Traceback (most recent call last):: /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 17:25:14.929957 2022] [cgi:error] [pid 6320] [client 127.0.0.1:53802] AH01215:   File "/home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py", line 12, in <module>: /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 17:25:14.929996 2022] [cgi:error] [pid 6320] [client 127.0.0.1:53802] AH01215:     modulo = __import__(PACKAGE, fromlist=[CONTROLLER]): /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 17:25:14.930050 2022] [cgi:error] [pid 6320] [client 127.0.0.1:53802] AH01215:   File "/home/daniel/Proyectos/MostPlayed/rootsystem/application/modules/song.py", line 110: /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 17:25:14.930080 2022] [cgi:error] [pid 6320] [client 127.0.0.1:53802] AH01215:     release_date=str(resultado[2]),: /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 17:25:14.930096 2022] [cgi:error] [pid 6320] [client 127.0.0.1:53802] AH01215:     ^: /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 17:25:14.930123 2022] [cgi:error] [pid 6320] [client 127.0.0.1:53802] AH01215: IndentationError: unexpected indent: /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 17:25:14.938451 2022] [cgi:error] [pid 6320] [client 127.0.0.1:53802] End of script output before headers: xfc.py
[Tue Jan 11 17:30:34.313241 2022] [cgi:error] [pid 6324] [client 127.0.0.1:53810] AH01215: Traceback (most recent call last):: /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 17:30:34.313404 2022] [cgi:error] [pid 6324] [client 127.0.0.1:53810] AH01215:   File "/home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py", line 17, in <module>: /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 17:30:34.313439 2022] [cgi:error] [pid 6324] [client 127.0.0.1:53810] AH01215:     getattr(controller, RESOURCE)(): /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 17:30:34.313518 2022] [cgi:error] [pid 6324] [client 127.0.0.1:53810] AH01215:   File "/home/daniel/Proyectos/MostPlayed/rootsystem/application/modules/song.py", line 94, in search_lookup: /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 17:30:34.313550 2022] [cgi:error] [pid 6324] [client 127.0.0.1:53810] AH01215:     show(dumps(resultado)): /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 17:30:34.313589 2022] [cgi:error] [pid 6324] [client 127.0.0.1:53810] AH01215: NameError: name 'dumps' is not defined: /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 17:30:34.334992 2022] [cgi:error] [pid 6324] [client 127.0.0.1:53810] End of script output before headers: xfc.py
[Tue Jan 11 17:33:32.567803 2022] [cgi:error] [pid 6320] [client 127.0.0.1:53820] AH01215: Traceback (most recent call last):: /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 17:33:32.567948 2022] [cgi:error] [pid 6320] [client 127.0.0.1:53820] AH01215:   File "/home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py", line 17, in <module>: /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 17:33:32.567977 2022] [cgi:error] [pid 6320] [client 127.0.0.1:53820] AH01215:     getattr(controller, RESOURCE)(): /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 17:33:32.568039 2022] [cgi:error] [pid 6320] [client 127.0.0.1:53820] AH01215:   File "/home/daniel/Proyectos/MostPlayed/rootsystem/application/modules/song.py", line 93, in search_lookup: /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 17:33:32.568077 2022] [cgi:error] [pid 6320] [client 127.0.0.1:53820] AH01215:     resultado = SongHelper().get_name(kword='lala'): /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 17:33:32.568168 2022] [cgi:error] [pid 6320] [client 127.0.0.1:53820] AH01215:   File "/home/daniel/Proyectos/MostPlayed/rootsystem/application/modules/song.py", line 107, in get_name: /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 17:33:32.568201 2022] [cgi:error] [pid 6320] [client 127.0.0.1:53820] AH01215:     resultado = DBQuery().execute(sql)[0]: /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 17:33:32.568250 2022] [cgi:error] [pid 6320] [client 127.0.0.1:53820] AH01215: IndexError: tuple index out of range: /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 17:33:32.590042 2022] [cgi:error] [pid 6320] [client 127.0.0.1:53820] End of script output before headers: xfc.py
[Tue Jan 11 18:14:34.137829 2022] [cgi:error] [pid 6322] [client 127.0.0.1:53932] AH01215: Traceback (most recent call last):: /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 18:14:34.138052 2022] [cgi:error] [pid 6322] [client 127.0.0.1:53932] AH01215:   File "/home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py", line 12, in <module>: /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 18:14:34.138095 2022] [cgi:error] [pid 6322] [client 127.0.0.1:53932] AH01215:     modulo = __import__(PACKAGE, fromlist=[CONTROLLER]): /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 18:14:34.138151 2022] [cgi:error] [pid 6322] [client 127.0.0.1:53932] AH01215:   File "/home/daniel/Proyectos/MostPlayed/rootsystem/application/modules/song.py", line 151: /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 18:14:34.138185 2022] [cgi:error] [pid 6322] [client 127.0.0.1:53932] AH01215:     data = Template(json).safe_substitute(dicc): /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 18:14:34.138199 2022] [cgi:error] [pid 6322] [client 127.0.0.1:53932] AH01215:     ^: /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 18:14:34.138227 2022] [cgi:error] [pid 6322] [client 127.0.0.1:53932] AH01215: IndentationError: unexpected indent: /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 18:14:34.146723 2022] [cgi:error] [pid 6322] [client 127.0.0.1:53932] End of script output before headers: xfc.py
[Tue Jan 11 18:21:18.211913 2022] [cgi:error] [pid 6321] [client 127.0.0.1:53956] AH01215: Traceback (most recent call last):: /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 18:21:18.212075 2022] [cgi:error] [pid 6321] [client 127.0.0.1:53956] AH01215:   File "/home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py", line 17, in <module>: /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 18:21:18.212120 2022] [cgi:error] [pid 6321] [client 127.0.0.1:53956] AH01215:     getattr(controller, RESOURCE)(): /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 18:21:18.212196 2022] [cgi:error] [pid 6321] [client 127.0.0.1:53956] AH01215:   File "/home/daniel/Proyectos/MostPlayed/rootsystem/application/modules/song.py", line 105, in get_top: /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 18:21:18.212251 2022] [cgi:error] [pid 6321] [client 127.0.0.1:53956] AH01215:     query = SongHelper().get_top(ARG) : /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 18:21:18.212326 2022] [cgi:error] [pid 6321] [client 127.0.0.1:53956] AH01215:   File "/home/daniel/Proyectos/MostPlayed/rootsystem/application/modules/song.py", line 154, in get_top: /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 18:21:18.212351 2022] [cgi:error] [pid 6321] [client 127.0.0.1:53956] AH01215:     lista.update(data): /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 18:21:18.212410 2022] [cgi:error] [pid 6321] [client 127.0.0.1:53956] AH01215: ValueError: dictionary update sequence element #0 has length 1; 2 is required: /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 18:21:18.243283 2022] [cgi:error] [pid 6321] [client 127.0.0.1:53956] End of script output before headers: xfc.py
[Tue Jan 11 18:30:27.723777 2022] [cgi:error] [pid 6322] [client 127.0.0.1:53998] AH01215: Traceback (most recent call last):: /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 18:30:27.723929 2022] [cgi:error] [pid 6322] [client 127.0.0.1:53998] AH01215:   File "/home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py", line 17, in <module>: /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 18:30:27.723959 2022] [cgi:error] [pid 6322] [client 127.0.0.1:53998] AH01215:     getattr(controller, RESOURCE)(): /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 18:30:27.724020 2022] [cgi:error] [pid 6322] [client 127.0.0.1:53998] AH01215:   File "/home/daniel/Proyectos/MostPlayed/rootsystem/application/modules/song.py", line 105, in get_top: /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 18:30:27.724051 2022] [cgi:error] [pid 6322] [client 127.0.0.1:53998] AH01215:     query = SongHelper().get_top(ARG) : /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 18:30:27.724112 2022] [cgi:error] [pid 6322] [client 127.0.0.1:53998] AH01215:   File "/home/daniel/Proyectos/MostPlayed/rootsystem/application/modules/song.py", line 155, in get_top: /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 18:30:27.724157 2022] [cgi:error] [pid 6322] [client 127.0.0.1:53998] AH01215:     return '\\n'.join([line.rstrip() for line in lista.splitlines()]): /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 18:30:27.724200 2022] [cgi:error] [pid 6322] [client 127.0.0.1:53998] AH01215: AttributeError: 'list' object has no attribute 'splitlines': /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Tue Jan 11 18:30:27.745809 2022] [cgi:error] [pid 6322] [client 127.0.0.1:53998] End of script output before headers: xfc.py
[Wed Jan 12 09:26:47.737896 2022] [cgi:error] [pid 1054] [client 127.0.0.1:50990] End of script output before headers: xfc.py
[Wed Jan 12 09:27:17.562801 2022] [cgi:error] [pid 3890] [client 127.0.0.1:50994] End of script output before headers: xfc.py
[Wed Jan 12 09:27:51.957652 2022] [cgi:error] [pid 1052] [client 127.0.0.1:51002] End of script output before headers: xfc.py
[Wed Jan 12 09:34:52.974553 2022] [cgi:error] [pid 1053] [client 127.0.0.1:51006] End of script output before headers: xfc.py
[Wed Jan 12 09:34:58.064019 2022] [cgi:error] [pid 1054] [client 127.0.0.1:51008] End of script output before headers: xfc.py
[Wed Jan 12 09:41:02.324888 2022] [cgi:error] [pid 3890] [client 127.0.0.1:51034] End of script output before headers: xfc.py
[Wed Jan 12 09:41:51.280626 2022] [cgi:error] [pid 1051] [client 127.0.0.1:51054] End of script output before headers: xfc.py
[Wed Jan 12 09:42:03.674414 2022] [cgi:error] [pid 1052] [client 127.0.0.1:51056] End of script output before headers: xfc.py
[Wed Jan 12 09:52:12.309009 2022] [cgi:error] [pid 1054] [client 127.0.0.1:51080] AH01215: Traceback (most recent call last):: /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Wed Jan 12 09:52:12.309158 2022] [cgi:error] [pid 1054] [client 127.0.0.1:51080] AH01215:   File "/home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py", line 17, in <module>: /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Wed Jan 12 09:52:12.309188 2022] [cgi:error] [pid 1054] [client 127.0.0.1:51080] AH01215:     getattr(controller, RESOURCE)(): /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Wed Jan 12 09:52:12.309246 2022] [cgi:error] [pid 1054] [client 127.0.0.1:51080] AH01215:   File "/home/daniel/Proyectos/MostPlayed/rootsystem/application/modules/song.py", line 85, in ver: /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Wed Jan 12 09:52:12.309273 2022] [cgi:error] [pid 1054] [client 127.0.0.1:51080] AH01215:     self.model.select(): /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Wed Jan 12 09:52:12.309371 2022] [cgi:error] [pid 1054] [client 127.0.0.1:51080] AH01215:   File "/home/daniel/Proyectos/MostPlayed/rootsystem/application/modules/song.py", line 47, in select: /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Wed Jan 12 09:52:12.309404 2022] [cgi:error] [pid 1054] [client 127.0.0.1:51080] AH01215:     resultados = DBQuery().execute(sql)[0]: /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Wed Jan 12 09:52:12.309435 2022] [cgi:error] [pid 1054] [client 127.0.0.1:51080] AH01215: IndexError: tuple index out of range: /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Wed Jan 12 09:52:12.331730 2022] [cgi:error] [pid 1054] [client 127.0.0.1:51080] End of script output before headers: xfc.py
[Wed Jan 12 10:43:02.166756 2022] [cgi:error] [pid 1052] [client 127.0.0.1:51268] AH01215: Traceback (most recent call last):: /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Wed Jan 12 10:43:02.166899 2022] [cgi:error] [pid 1052] [client 127.0.0.1:51268] AH01215:   File "/home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py", line 12, in <module>: /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Wed Jan 12 10:43:02.166938 2022] [cgi:error] [pid 1052] [client 127.0.0.1:51268] AH01215:     modulo = __import__(PACKAGE, fromlist=[CONTROLLER]): /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Wed Jan 12 10:43:02.167001 2022] [cgi:error] [pid 1052] [client 127.0.0.1:51268] AH01215:   File "/home/daniel/Proyectos/MostPlayed/rootsystem/application/modules/song.py", line 6, in <module>: /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Wed Jan 12 10:43:02.167035 2022] [cgi:error] [pid 1052] [client 127.0.0.1:51268] AH01215:     from modules.genresong import GenreSong: /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Wed Jan 12 10:43:02.167097 2022] [cgi:error] [pid 1052] [client 127.0.0.1:51268] AH01215:   File "/home/daniel/Proyectos/MostPlayed/rootsystem/application/modules/genresong.py", line 2, in <module>: /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Wed Jan 12 10:43:02.167127 2022] [cgi:error] [pid 1052] [client 127.0.0.1:51268] AH01215:     from modules.genre import Genre: /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Wed Jan 12 10:43:02.167189 2022] [cgi:error] [pid 1052] [client 127.0.0.1:51268] AH01215:   File "/home/daniel/Proyectos/MostPlayed/rootsystem/application/modules/genre.py", line 27, in <module>: /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Wed Jan 12 10:43:02.167214 2022] [cgi:error] [pid 1052] [client 127.0.0.1:51268] AH01215:     class GenreView(self):: /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Wed Jan 12 10:43:02.167246 2022] [cgi:error] [pid 1052] [client 127.0.0.1:51268] AH01215: NameError: name 'self' is not defined: /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Wed Jan 12 10:43:02.187672 2022] [cgi:error] [pid 1052] [client 127.0.0.1:51268] End of script output before headers: xfc.py
[Wed Jan 12 10:55:40.401466 2022] [cgi:error] [pid 1051] [client 127.0.0.1:51360] AH01215: Traceback (most recent call last):: /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Wed Jan 12 10:55:40.401628 2022] [cgi:error] [pid 1051] [client 127.0.0.1:51360] AH01215:   File "/home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py", line 17, in <module>: /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Wed Jan 12 10:55:40.401900 2022] [cgi:error] [pid 1051] [client 127.0.0.1:51360] AH01215:     getattr(controller, RESOURCE)(): /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Wed Jan 12 10:55:40.401960 2022] [cgi:error] [pid 1051] [client 127.0.0.1:51360] AH01215:   File "/home/daniel/Proyectos/MostPlayed/rootsystem/application/modules/song.py", line 91, in ver: /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Wed Jan 12 10:55:40.401982 2022] [cgi:error] [pid 1051] [client 127.0.0.1:51360] AH01215:     self.model.select(): /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Wed Jan 12 10:55:40.402059 2022] [cgi:error] [pid 1051] [client 127.0.0.1:51360] AH01215:   File "/home/daniel/Proyectos/MostPlayed/rootsystem/application/modules/song.py", line 63, in select: /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Wed Jan 12 10:55:40.402079 2022] [cgi:error] [pid 1051] [client 127.0.0.1:51360] AH01215:     cl.select(): /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Wed Jan 12 10:55:40.402139 2022] [cgi:error] [pid 1051] [client 127.0.0.1:51360] AH01215:   File "/home/daniel/Proyectos/MostPlayed/rootsystem/application/modules/genresong.py", line 31, in select: /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Wed Jan 12 10:55:40.402161 2022] [cgi:error] [pid 1051] [client 127.0.0.1:51360] AH01215:     genero = Genre(): /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Wed Jan 12 10:55:40.402224 2022] [cgi:error] [pid 1051] [client 127.0.0.1:51360] AH01215:   File "/home/daniel/Proyectos/MostPlayed/rootsystem/application/modules/genre.py", line 14, in __init__: /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Wed Jan 12 10:55:40.402246 2022] [cgi:error] [pid 1051] [client 127.0.0.1:51360] AH01215:     self.genre_id = 0: /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Wed Jan 12 10:55:40.402278 2022] [cgi:error] [pid 1051] [client 127.0.0.1:51360] AH01215: NameError: name 'self' is not defined: /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Wed Jan 12 10:55:40.423540 2022] [cgi:error] [pid 1051] [client 127.0.0.1:51360] End of script output before headers: xfc.py
[Wed Jan 12 10:59:21.896336 2022] [cgi:error] [pid 1055] [client 127.0.0.1:51378] AH01215: Traceback (most recent call last):: /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Wed Jan 12 10:59:21.896541 2022] [cgi:error] [pid 1055] [client 127.0.0.1:51378] AH01215:   File "/home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py", line 17, in <module>: /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Wed Jan 12 10:59:21.896645 2022] [cgi:error] [pid 1055] [client 127.0.0.1:51378] AH01215:     getattr(controller, RESOURCE)(): /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Wed Jan 12 10:59:21.896735 2022] [cgi:error] [pid 1055] [client 127.0.0.1:51378] AH01215:   File "/home/daniel/Proyectos/MostPlayed/rootsystem/application/modules/song.py", line 99, in ver: /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Wed Jan 12 10:59:21.896778 2022] [cgi:error] [pid 1055] [client 127.0.0.1:51378] AH01215:     show_json(dumps(vars(self.model))): /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Wed Jan 12 10:59:21.896844 2022] [cgi:error] [pid 1055] [client 127.0.0.1:51378] AH01215:   File "/usr/lib/python3.7/json/__init__.py", line 231, in dumps: /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Wed Jan 12 10:59:21.896893 2022] [cgi:error] [pid 1055] [client 127.0.0.1:51378] AH01215:     return _default_encoder.encode(obj): /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Wed Jan 12 10:59:21.896958 2022] [cgi:error] [pid 1055] [client 127.0.0.1:51378] AH01215:   File "/usr/lib/python3.7/json/encoder.py", line 199, in encode: /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Wed Jan 12 10:59:21.897011 2022] [cgi:error] [pid 1055] [client 127.0.0.1:51378] AH01215:     chunks = self.iterencode(o, _one_shot=True): /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Wed Jan 12 10:59:21.897079 2022] [cgi:error] [pid 1055] [client 127.0.0.1:51378] AH01215:   File "/usr/lib/python3.7/json/encoder.py", line 257, in iterencode: /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Wed Jan 12 10:59:21.897117 2022] [cgi:error] [pid 1055] [client 127.0.0.1:51378] AH01215:     return _iterencode(o, 0): /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Wed Jan 12 10:59:21.897214 2022] [cgi:error] [pid 1055] [client 127.0.0.1:51378] AH01215:   File "/usr/lib/python3.7/json/encoder.py", line 179, in default: /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Wed Jan 12 10:59:21.897276 2022] [cgi:error] [pid 1055] [client 127.0.0.1:51378] AH01215:     raise TypeError(f'Object of type {o.__class__.__name__} ': /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Wed Jan 12 10:59:21.897335 2022] [cgi:error] [pid 1055] [client 127.0.0.1:51378] AH01215: TypeError: Object of type bytes is not JSON serializable: /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Wed Jan 12 10:59:21.922355 2022] [cgi:error] [pid 1055] [client 127.0.0.1:51378] End of script output before headers: xfc.py
[Wed Jan 12 11:04:46.825851 2022] [cgi:error] [pid 1051] [client 127.0.0.1:51400] AH01215: Traceback (most recent call last):: /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Wed Jan 12 11:04:46.826009 2022] [cgi:error] [pid 1051] [client 127.0.0.1:51400] AH01215:   File "/home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py", line 17, in <module>: /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Wed Jan 12 11:04:46.826045 2022] [cgi:error] [pid 1051] [client 127.0.0.1:51400] AH01215:     getattr(controller, RESOURCE)(): /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Wed Jan 12 11:04:46.826119 2022] [cgi:error] [pid 1051] [client 127.0.0.1:51400] AH01215:   File "/home/daniel/Proyectos/MostPlayed/rootsystem/application/modules/song.py", line 100, in ver: /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Wed Jan 12 11:04:46.826147 2022] [cgi:error] [pid 1051] [client 127.0.0.1:51400] AH01215:     show_json(dumps(d)): /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Wed Jan 12 11:04:46.826196 2022] [cgi:error] [pid 1051] [client 127.0.0.1:51400] AH01215:   File "/usr/lib/python3.7/json/__init__.py", line 231, in dumps: /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Wed Jan 12 11:04:46.826235 2022] [cgi:error] [pid 1051] [client 127.0.0.1:51400] AH01215:     return _default_encoder.encode(obj): /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Wed Jan 12 11:04:46.826289 2022] [cgi:error] [pid 1051] [client 127.0.0.1:51400] AH01215:   File "/usr/lib/python3.7/json/encoder.py", line 199, in encode: /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Wed Jan 12 11:04:46.826333 2022] [cgi:error] [pid 1051] [client 127.0.0.1:51400] AH01215:     chunks = self.iterencode(o, _one_shot=True): /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Wed Jan 12 11:04:46.826389 2022] [cgi:error] [pid 1051] [client 127.0.0.1:51400] AH01215:   File "/usr/lib/python3.7/json/encoder.py", line 257, in iterencode: /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Wed Jan 12 11:04:46.826421 2022] [cgi:error] [pid 1051] [client 127.0.0.1:51400] AH01215:     return _iterencode(o, 0): /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Wed Jan 12 11:04:46.826473 2022] [cgi:error] [pid 1051] [client 127.0.0.1:51400] AH01215:   File "/usr/lib/python3.7/json/encoder.py", line 179, in default: /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Wed Jan 12 11:04:46.826525 2022] [cgi:error] [pid 1051] [client 127.0.0.1:51400] AH01215:     raise TypeError(f'Object of type {o.__class__.__name__} ': /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Wed Jan 12 11:04:46.826574 2022] [cgi:error] [pid 1051] [client 127.0.0.1:51400] AH01215: TypeError: Object of type bytes is not JSON serializable: /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Wed Jan 12 11:04:46.848088 2022] [cgi:error] [pid 1051] [client 127.0.0.1:51400] End of script output before headers: xfc.py
[Wed Jan 12 11:05:24.760946 2022] [cgi:error] [pid 1053] [client 127.0.0.1:51404] AH01215: Traceback (most recent call last):: /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Wed Jan 12 11:05:24.761145 2022] [cgi:error] [pid 1053] [client 127.0.0.1:51404] AH01215:   File "/home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py", line 17, in <module>: /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Wed Jan 12 11:05:24.761219 2022] [cgi:error] [pid 1053] [client 127.0.0.1:51404] AH01215:     getattr(controller, RESOURCE)(): /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Wed Jan 12 11:05:24.761306 2022] [cgi:error] [pid 1053] [client 127.0.0.1:51404] AH01215:   File "/home/daniel/Proyectos/MostPlayed/rootsystem/application/modules/song.py", line 100, in ver: /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Wed Jan 12 11:05:24.761339 2022] [cgi:error] [pid 1053] [client 127.0.0.1:51404] AH01215:     show_json(loads(d)): /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Wed Jan 12 11:05:24.761396 2022] [cgi:error] [pid 1053] [client 127.0.0.1:51404] AH01215:   File "/usr/lib/python3.7/json/__init__.py", line 341, in loads: /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Wed Jan 12 11:05:24.761459 2022] [cgi:error] [pid 1053] [client 127.0.0.1:51404] AH01215:     raise TypeError(f'the JSON object must be str, bytes or bytearray, ': /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Wed Jan 12 11:05:24.761519 2022] [cgi:error] [pid 1053] [client 127.0.0.1:51404] AH01215: TypeError: the JSON object must be str, bytes or bytearray, not dict: /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Wed Jan 12 11:05:24.785118 2022] [cgi:error] [pid 1053] [client 127.0.0.1:51404] End of script output before headers: xfc.py
[Wed Jan 12 11:09:39.060518 2022] [cgi:error] [pid 1055] [client 127.0.0.1:51414] AH01215: Traceback (most recent call last):: /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Wed Jan 12 11:09:39.060703 2022] [cgi:error] [pid 1055] [client 127.0.0.1:51414] AH01215:   File "/home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py", line 17, in <module>: /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Wed Jan 12 11:09:39.060749 2022] [cgi:error] [pid 1055] [client 127.0.0.1:51414] AH01215:     getattr(controller, RESOURCE)(): /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Wed Jan 12 11:09:39.060818 2022] [cgi:error] [pid 1055] [client 127.0.0.1:51414] AH01215:   File "/home/daniel/Proyectos/MostPlayed/rootsystem/application/modules/song.py", line 100, in ver: /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Wed Jan 12 11:09:39.060858 2022] [cgi:error] [pid 1055] [client 127.0.0.1:51414] AH01215:     show_json(APIRESTFul(self.model).json_data): /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Wed Jan 12 11:09:39.060941 2022] [cgi:error] [pid 1055] [client 127.0.0.1:51414] AH01215:   File "/home/daniel/Proyectos/MostPlayed/rootsystem/application/core/api.py", line 7, in __init__: /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Wed Jan 12 11:09:39.060981 2022] [cgi:error] [pid 1055] [client 127.0.0.1:51414] AH01215:     self.json_data = self.dict2json(vars(obj)): /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Wed Jan 12 11:09:39.061049 2022] [cgi:error] [pid 1055] [client 127.0.0.1:51414] AH01215:   File "/home/daniel/Proyectos/MostPlayed/rootsystem/application/core/api.py", line 21, in dict2json: /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Wed Jan 12 11:09:39.061098 2022] [cgi:error] [pid 1055] [client 127.0.0.1:51414] AH01215:     obj = json.dumps(self.obj2dict(obj),indent=4, sort_keys=True): /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Wed Jan 12 11:09:39.061166 2022] [cgi:error] [pid 1055] [client 127.0.0.1:51414] AH01215:   File "/home/daniel/Proyectos/MostPlayed/rootsystem/application/core/api.py", line 14, in obj2dict: /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Wed Jan 12 11:09:39.061195 2022] [cgi:error] [pid 1055] [client 127.0.0.1:51414] AH01215:     obj[propiedad] = vars(valor): /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Wed Jan 12 11:09:39.061270 2022] [cgi:error] [pid 1055] [client 127.0.0.1:51414] AH01215: TypeError: vars() argument must have __dict__ attribute: /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Wed Jan 12 11:09:39.085038 2022] [cgi:error] [pid 1055] [client 127.0.0.1:51414] End of script output before headers: xfc.py
[Wed Jan 12 15:28:58.621290 2022] [cgi:error] [pid 1005] [client 127.0.0.1:45324] AH01215: Traceback (most recent call last):: /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Wed Jan 12 15:28:58.969372 2022] [cgi:error] [pid 1005] [client 127.0.0.1:45324] AH01215:   File "/home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py", line 17, in <module>: /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Wed Jan 12 15:28:58.969449 2022] [cgi:error] [pid 1005] [client 127.0.0.1:45324] AH01215:     getattr(controller, RESOURCE)(): /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Wed Jan 12 15:28:58.969560 2022] [cgi:error] [pid 1005] [client 127.0.0.1:45324] AH01215:   File "/home/daniel/Proyectos/MostPlayed/rootsystem/application/modules/song.py", line 136, in get_by_genre: /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Wed Jan 12 15:28:58.969615 2022] [cgi:error] [pid 1005] [client 127.0.0.1:45324] AH01215:     query = GenreSongHelper().get_by_genre(): /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Wed Jan 12 15:28:58.969715 2022] [cgi:error] [pid 1005] [client 127.0.0.1:45324] AH01215: AttributeError: 'GenreSongHelper' object has no attribute 'get_by_genre': /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Wed Jan 12 15:28:58.969779 2022] [cgi:error] [pid 1005] [client 127.0.0.1:45324] End of script output before headers: xfc.py
[Wed Jan 12 15:29:49.990404 2022] [cgi:error] [pid 1006] [client 127.0.0.1:45326] AH01215: Traceback (most recent call last):: /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Wed Jan 12 15:29:49.990553 2022] [cgi:error] [pid 1006] [client 127.0.0.1:45326] AH01215:   File "/home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py", line 17, in <module>: /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Wed Jan 12 15:29:49.990583 2022] [cgi:error] [pid 1006] [client 127.0.0.1:45326] AH01215:     getattr(controller, RESOURCE)(): /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Wed Jan 12 15:29:49.990647 2022] [cgi:error] [pid 1006] [client 127.0.0.1:45326] AH01215:   File "/home/daniel/Proyectos/MostPlayed/rootsystem/application/modules/song.py", line 136, in get_by_genre: /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Wed Jan 12 15:29:49.990685 2022] [cgi:error] [pid 1006] [client 127.0.0.1:45326] AH01215:     query = GenreSongHelper().group_by_genre(): /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Wed Jan 12 15:29:49.990750 2022] [cgi:error] [pid 1006] [client 127.0.0.1:45326] AH01215:   File "/home/daniel/Proyectos/MostPlayed/rootsystem/application/modules/genresong.py", line 42, in group_by_genre: /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Wed Jan 12 15:29:49.990781 2022] [cgi:error] [pid 1006] [client 127.0.0.1:45326] AH01215:     resultado = DBQuery().execute(sql): /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Wed Jan 12 15:29:49.990840 2022] [cgi:error] [pid 1006] [client 127.0.0.1:45326] AH01215:   File "/home/daniel/Proyectos/MostPlayed/rootsystem/application/core/db.py", line 17, in execute: /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Wed Jan 12 15:29:49.990865 2022] [cgi:error] [pid 1006] [client 127.0.0.1:45326] AH01215:     cursor.execute(self.sql): /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Wed Jan 12 15:29:49.990919 2022] [cgi:error] [pid 1006] [client 127.0.0.1:45326] AH01215:   File "/usr/lib/python3/dist-packages/MySQLdb/cursors.py", line 250, in execute: /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Wed Jan 12 15:29:49.990956 2022] [cgi:error] [pid 1006] [client 127.0.0.1:45326] AH01215:     self.errorhandler(self, exc, value): /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Wed Jan 12 15:29:49.991057 2022] [cgi:error] [pid 1006] [client 127.0.0.1:45326] AH01215:   File "/usr/lib/python3/dist-packages/MySQLdb/connections.py", line 50, in defaulterrorhandler: /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Wed Jan 12 15:29:49.991079 2022] [cgi:error] [pid 1006] [client 127.0.0.1:45326] AH01215:     raise errorvalue: /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Wed Jan 12 15:29:49.991128 2022] [cgi:error] [pid 1006] [client 127.0.0.1:45326] AH01215:   File "/usr/lib/python3/dist-packages/MySQLdb/cursors.py", line 247, in execute: /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Wed Jan 12 15:29:49.991154 2022] [cgi:error] [pid 1006] [client 127.0.0.1:45326] AH01215:     res = self._query(query): /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Wed Jan 12 15:29:49.991205 2022] [cgi:error] [pid 1006] [client 127.0.0.1:45326] AH01215:   File "/usr/lib/python3/dist-packages/MySQLdb/cursors.py", line 411, in _query: /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Wed Jan 12 15:29:49.991233 2022] [cgi:error] [pid 1006] [client 127.0.0.1:45326] AH01215:     rowcount = self._do_query(q): /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Wed Jan 12 15:29:49.991285 2022] [cgi:error] [pid 1006] [client 127.0.0.1:45326] AH01215:   File "/usr/lib/python3/dist-packages/MySQLdb/cursors.py", line 374, in _do_query: /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Wed Jan 12 15:29:49.991305 2022] [cgi:error] [pid 1006] [client 127.0.0.1:45326] AH01215:     db.query(q): /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Wed Jan 12 15:29:49.991354 2022] [cgi:error] [pid 1006] [client 127.0.0.1:45326] AH01215:   File "/usr/lib/python3/dist-packages/MySQLdb/connections.py", line 292, in query: /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Wed Jan 12 15:29:49.991385 2022] [cgi:error] [pid 1006] [client 127.0.0.1:45326] AH01215:     _mysql.connection.query(self, query): /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Wed Jan 12 15:29:49.991439 2022] [cgi:error] [pid 1006] [client 127.0.0.1:45326] AH01215: _mysql_exceptions.OperationalError: (1054, "Unknown column 'renre' in 'field list'"): /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Wed Jan 12 15:29:50.013054 2022] [cgi:error] [pid 1006] [client 127.0.0.1:45326] End of script output before headers: xfc.py
[Wed Jan 12 16:04:04.919955 2022] [cgi:error] [pid 1007] [client 127.0.0.1:45458] AH01215: Traceback (most recent call last):: /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Wed Jan 12 16:04:04.920161 2022] [cgi:error] [pid 1007] [client 127.0.0.1:45458] AH01215:   File "/home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py", line 17, in <module>: /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Wed Jan 12 16:04:04.920218 2022] [cgi:error] [pid 1007] [client 127.0.0.1:45458] AH01215:     getattr(controller, RESOURCE)(): /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Wed Jan 12 16:04:04.920337 2022] [cgi:error] [pid 1007] [client 127.0.0.1:45458] AH01215:   File "/home/daniel/Proyectos/MostPlayed/rootsystem/application/modules/song.py", line 141, in add_track: /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Wed Jan 12 16:04:04.920387 2022] [cgi:error] [pid 1007] [client 127.0.0.1:45458] AH01215:     json = form['data'].value: /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Wed Jan 12 16:04:04.920463 2022] [cgi:error] [pid 1007] [client 127.0.0.1:45458] AH01215:   File "/usr/lib/python3.7/cgi.py", line 534, in __getitem__: /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Wed Jan 12 16:04:04.920507 2022] [cgi:error] [pid 1007] [client 127.0.0.1:45458] AH01215:     raise KeyError(key): /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Wed Jan 12 16:04:04.920577 2022] [cgi:error] [pid 1007] [client 127.0.0.1:45458] AH01215: KeyError: 'data': /home/daniel/Proyectos/MostPlayed/rootsystem/application/xfc.py
[Wed Jan 12 16:04:04.942189 2022] [cgi:error] [pid 1007] [client 127.0.0.1:45458] End of script output before headers: xfc.py
